<!DOCTYPE html>

<head>
  <meta charset="utf-8" />
  <title>Kibs | Africa's BioTech Company</title>
  <meta
    content="Your Trusted Partner in Biomedical Research will modernize healthcare in the 21st century. Our focus lies in creating innovative therapeutics designed to prevent diseases and lengthen healthspan—the period of life spent in good health."
    name="description" />
  <meta content="Kibs Bio | Africa's BioTech Company" property="og:title" />
  <meta
    content="Your Trusted Partner in Biomedical Research will modernize healthcare in the 21st century. Our focus lies in creating innovative therapeutics designed to prevent diseases and lengthen healthspan—the period of life spent in good health."
    property="og:description" />
  <meta content="Kibs Bio | Africa's BioTech Company" property="twitter:title" />


  <meta property="og:type" content="website" />

  <meta content="summary_large_image" name="twitter:card" />
  <meta content="width=device-width, initial-scale=1" name="viewport" />

  <!-- IMPORTANT STYLESHEET FOR THE PAGE -CAMBRIAN -->

  <link href="https://cdn.prod.website-files.com/600be113c4111047016de64a/css/cambrian-dev.webflow.0733b0f40.min.css"
    rel="stylesheet" type="text/css" />


  <script type="text/javascript">
    ! function (o, c) {
      var n = c.documentElement,
        t = " w-mod-";
      n.className += t + "js", ("ontouchstart" in o || o.DocumentTouch && c instanceof DocumentTouch) && (n
        .className += t + "touch")
    }(window, document);
  </script>


  <link href="https://i0.wp.com/www.kibs.co.ke/wp/wp-content/uploads/2023/10/kibs-logo-02.png?fit=120,85&amp;ssl=1"
    rel="shortcut icon" type="image/x-icon" />


  <script type="text/javascript">
    window.dataLayer = window.dataLayer || [];

    function gtag() {
      dataLayer.push(arguments);
    }
    gtag('js', new Date());
    gtag('config', 'G-Y7NNGFHMFL', {
      'anonymize_ip': false
    });
  </script>

  <meta name="pinterest" content="nopin" />
  <meta name="twitter:card" content="summary_large_image">

  <!-- CUSTOM STYLE -->
  <style>
    html {
      -webkit-font-smoothing: antialiased;
      -moz-font-smoothing: antialiased;
      -o-font-smoothing: antialiased
    }


    input,
    select,
    textarea {
      appearance: none;
      -moz-appearance: none;
      -webkit-appearance: none
    }

    @media screen and (max-width:767px) {
      .modal {
        display: block
      }

      .modal .c-660 {
        height: -webkit-fill-available
      }

      .modal .abs-shadow-card {
        display: none
      }

      .modal h3 {
        font-size: 1.75em
      }
    }
  </style>
  <!-- END CUSTOM STYLE -->



  <style>
    .s.nav {
      background-color: #081B42
    }
  </style>


  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet"
    integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">



  <script>
    (function (i, s, o, g, r, a, m) {
      i['GoogleAnalyticsObject'] = r;
      i[r] = i[r] || function () {
        (i[r].q = i[r].q || []).push(arguments)
      }, i[r].l = 1 * new Date();
      a = s.createElement(o),
        m = s.getElementsByTagName(o)[0];
      a.async = 1;
      a.src = g;
      m.parentNode.insertBefore(a, m)
    })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');

    ga('create', 'UA-47276536-5', 'auto');
    ga('send', 'pageview');
  </script>

  <!-- Google Tag Manager -->

  <script>
    (function (w, d, s, l, i) {
      w[l] = w[l] || [];
      w[l].push({
        'gtm.start': new Date().getTime(),
        event: 'gtm.js'
      });
      var f = d.getElementsByTagName(s)[0],
        j = d.createElement(s),
        dl = l != 'dataLayer' ? '&l=' + l : '';
      j.async = true;
      j.src =
        'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
      f.parentNode.insertBefore(j, f);
    })(window, document, 'script', 'dataLayer', 'GTM-5KHQW33');
  </script>


  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0" />
  <!-- <link rel="shortcut icon" href="/wp-content/themes/whatfix-blog/favicon-updated2.ico" /> -->

  <style>
    img:is([sizes="auto"i], [sizes^="auto,"i]) {
      contain-intrinsic-size: 3000px 1500px
    }
  </style>



  <style id='classic-theme-styles-inline-css' type='text/css'>
    /*! This file is auto-generated */
    .wp-block-button__link {
      color: #fff;
      background-color: #32373c;
      border-radius: 9999px;
      box-shadow: none;
      text-decoration: none;
      padding: calc(.667em + 2px) calc(1.333em + 2px);
      font-size: 1.125em
    }

    .wp-block-file__button {
      background: #32373c;
      color: #fff;
      text-decoration: none
    }
  </style>




  <!-- THE LINK THAT IS CAUSING TROUBLE WITH MY HEADER -->
  <!-- 
  
  <link rel='stylesheet' id='style-css'
    href='../wp-content/themes/wfx-blog-new-theme/assets/css/style.css%3Fver=1660820369.css' type='text/css'
    media='screen' /> -->


  <link rel='stylesheet' id='WhatfixStyle-css'
    href='../wp-content/themes/wfx-blog-new-theme/assets/css/WhatfixStyle.css%3Fver=1737095396.css' type='text/css'
    media='screen' />
  <link rel='stylesheet' id='flickity-css-css'
    href='https://cdnjs.cloudflare.com/ajax/libs/flickity/2.3.0/flickity.min.css?ver=2.3.0' type='text/css'
    media='all' />
  <link rel='stylesheet' id='elementor-frontend-css'
    href='../wp-content/plugins/elementor/assets/css/frontend.min.css%3Fver=3.25.11.css' type='text/css' media='all' />
  <link rel='stylesheet' id='widget-post-info-css'
    href='../wp-content/plugins/elementor-pro/assets/css/widget-post-info.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-icon-list-css'
    href='../wp-content/plugins/elementor/assets/css/widget-icon-list.min.css%3Fver=3.25.11.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-heading-css'
    href='../wp-content/plugins/elementor/assets/css/widget-heading.min.css%3Fver=3.25.11.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-table-of-contents-css'
    href='../wp-content/plugins/elementor-pro/assets/css/widget-table-of-contents.min.css%3Fver=3.25.5.css'
    type='text/css' media='all' />
  <link rel='stylesheet' id='widget-share-buttons-css'
    href='../wp-content/plugins/elementor-pro/assets/css/widget-share-buttons.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='e-apple-webkit-css'
    href='../wp-content/plugins/elementor/assets/css/conditionals/apple-webkit.min.css%3Fver=3.25.11.css'
    type='text/css' media='all' />
  <link rel='stylesheet' id='widget-hotspot-css'
    href='../wp-content/plugins/elementor-pro/assets/css/widget-hotspot.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-toggle-css'
    href='../wp-content/plugins/elementor/assets/css/widget-toggle.min.css%3Fver=3.25.11.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='e-animation-grow-css'
    href='../wp-content/plugins/elementor/assets/lib/animations/styles/e-animation-grow.min.css%3Fver=3.25.11.css'
    type='text/css' media='all' />
  <link rel='stylesheet' id='widget-form-css'
    href='../wp-content/plugins/elementor-pro/assets/css/widget-form.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-posts-css'
    href='../wp-content/plugins/elementor-pro/assets/css/widget-posts.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='swiper-css'
    href='../wp-content/plugins/elementor/assets/lib/swiper/v8/css/swiper.min.css%3Fver=8.4.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='e-swiper-css'
    href='../wp-content/plugins/elementor/assets/css/conditionals/e-swiper.min.css%3Fver=3.25.11.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='elementor-post-13020-css'
    href='../wp-content/uploads/elementor/css/post-13020.css%3Fver=1734008163.css' type='text/css' media='all' />
  <link rel='stylesheet' id='e-popup-style-css'
    href='../wp-content/plugins/elementor-pro/assets/css/conditionals/popup.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-nav-menu-css'
    href='../wp-content/plugins/elementor-pro/assets/css/widget-nav-menu.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-author-box-css'
    href='../wp-content/plugins/elementor-pro/assets/css/widget-author-box.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-image-css'
    href='../wp-content/plugins/elementor/assets/css/widget-image.min.css%3Fver=3.25.11.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-text-editor-css'
    href='../wp-content/plugins/elementor/assets/css/widget-text-editor.min.css%3Fver=3.25.11.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-call-to-action-css'
    href='../wp-content/plugins/elementor-pro/assets/css/widget-call-to-action.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='e-transitions-css'
    href='../wp-content/plugins/elementor-pro/assets/css/conditionals/transitions.min.css%3Fver=3.25.5.css'
    type='text/css' media='all' />
  <link rel='stylesheet' id='widget-testimonial-css'
    href='../wp-content/plugins/elementor/assets/css/widget-testimonial.min.css%3Fver=3.25.11.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='widget-divider-css'
    href='../wp-content/plugins/elementor/assets/css/widget-divider.min.css%3Fver=3.25.11.css' type='text/css'
    media='all' />
  <link rel='stylesheet' id='e-animation-push-css'
    href='../wp-content/plugins/elementor/assets/lib/animations/styles/e-animation-push.min.css%3Fver=3.25.11.css'
    type='text/css' media='all' />
  <link rel='stylesheet' id='elementor-post-83318-css'
    href='../wp-content/uploads/elementor/css/post-83318.css%3Fver=1737560777.css' type='text/css' media='all' />
  <link rel='stylesheet' id='google-fonts-1-css'
    href='https://fonts.googleapis.com/css?family=DM+Sans%3A100%2C100italic%2C200%2C200italic%2C300%2C300italic%2C400%2C400italic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%2C900%2C900italic%7CRoboto+Slab%3A100%2C100italic%2C200%2C200italic%2C300%2C300italic%2C400%2C400italic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%2C900%2C900italic%7CRoboto%3A100%2C100italic%2C200%2C200italic%2C300%2C300italic%2C400%2C400italic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%2C900%2C900italic%7CRubik%3A100%2C100italic%2C200%2C200italic%2C300%2C300italic%2C400%2C400italic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%2C900%2C900italic%7CIBM+Plex+Sans%3A100%2C100italic%2C200%2C200italic%2C300%2C300italic%2C400%2C400italic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%2C900%2C900italic%7CIBM+Plex+Mono%3A100%2C100italic%2C200%2C200italic%2C300%2C300italic%2C400%2C400italic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%2C900%2C900italic&#038;display=swap&#038;ver=6.7.1'
    type='text/css' media='all' />
  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
  <script type="text/javascript" src="../wp-includes/js/jquery/jquery.min.js%3Fver=3.7.1" id="jquery-core-js"></script>
  <link rel="https://api.w.org/" href="https://whatfix.com/blog/wp-json/" />
  <link rel="alternate" title="JSON" type="application/json"
    href="https://whatfix.com/blog/wp-json/wp/v2/posts/15822" />
  <link rel="EditURI" type="application/rsd+xml" title="RSD" href="https://whatfix.com/blog/xmlrpc.php?rsd" />
  <meta name="generator" content="WordPress 6.7.1" />
  <link rel='shortlink' href='https://whatfix.com/blog/?p=15822' />
  <link rel="alternate" title="oEmbed (JSON)" type="application/json+oembed"
    href="https://whatfix.com/blog/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fwhatfix.com%2Fblog%2Ftypes-employee-training-programs%2F" />
  <link rel="alternate" title="oEmbed (XML)" type="text/xml+oembed"
    href="https://whatfix.com/blog/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fwhatfix.com%2Fblog%2Ftypes-employee-training-programs%2F&#038;format=xml" />
  <script>
    document.documentElement.className = document.documentElement.className.replace('no-js', 'js');
  </script>
  <style>
    .no-js img.lazyload {
      display: none;
    }

    figure.wp-block-image img.lazyloading {
      min-width: 150px;
    }

    .lazyload,
    .lazyloading {
      opacity: 0;
    }

    .lazyloaded {
      opacity: 1;
      transition: opacity 100ms;
      transition-delay: 0ms;
    }
  </style>
  <!-- <meta name="generator"
          content="Elementor 3.25.11; features: e_font_icon_svg, e_optimized_control_loading; settings: css_print_method-external, google_font-enabled, font_display-swap">
        -->

  <style>
    .e-con.e-parent:nth-of-type(n+4):not(.e-lazyloaded):not(.e-no-lazyload),
    .e-con.e-parent:nth-of-type(n+4):not(.e-lazyloaded):not(.e-no-lazyload) * {
      background-image: none !important;
    }

    @media screen and (max-height: 1024px) {

      .e-con.e-parent:nth-of-type(n+3):not(.e-lazyloaded):not(.e-no-lazyload),
      .e-con.e-parent:nth-of-type(n+3):not(.e-lazyloaded):not(.e-no-lazyload) * {
        background-image: none !important;
      }
    }

    @media screen and (max-height: 640px) {

      .e-con.e-parent:nth-of-type(n+2):not(.e-lazyloaded):not(.e-no-lazyload),
      .e-con.e-parent:nth-of-type(n+2):not(.e-lazyloaded):not(.e-no-lazyload) * {
        background-image: none !important;
      }
    }
  </style>


  <script>
    var hsscript = document.createElement("script");
    hsscript.id = "wphs";
    hsscript.src = "https://cdn.jsdelivr.net/npm/hockeystack@latest/hockeystack.min.js";
    hsscript.async = 1;
    hsscript.dataset.apikey = "4703ad98081b8f593831188ed22c1a";
    hsscript.dataset.cookieless = 1;
    hsscript.dataset.autoIdentify = 1;
    document.getElementsByTagName('head')[0].append(hsscript);
  </script>



</head>

<body>

  <div class="no w-embed">
    <style>
      .notif-wrap~.nav {
        position: relative !important;
        transform: none !important
      }

      .nav-wrap.with-notif {
        transform: none !important
      }

      input:valid {
        border-color: #36c
      }

      .w-form-done p {
        font-weight: 500 !important
      }

      .w-form-fail {
        padding: 1em;
        border-radius: 10px;
        font-size: 1em;
        text-align: center;
        margin-top: 1em;
        font-weight: 400;
        letter-spacing: -0.02em
      }

      .row:first-child,
      .row-team:first-child,
      .pipeline .w-dyn-item:first-child .grid,
      .rt.preview p:first-child {
        margin-top: 0
      }

      .link-social:last-child {
        margin-right: 0
      }

      .tr {
        pointer-events: none
      }

      .btn:not(.btn-blue):not(.btn-maroon):not(.btn-upload) {
        background: linear-gradient(90deg, #109E5A 0%, #3366CC 100%)
      }

      .abs svg {
        position: absolute;
        left: 0;
        top: 0;
        right: 0;
        bottom: 0;
        width: 100%;
        height: auto
      }

      /* .abs.abs-bg-bot svg {
                top: auto
            } */

      /* .abs.abs-footer svg,
            .c-404 svg {
                height: 100%
            } */

      .s-404 {
        max-height: -webkit-fill-available
      }

      .cta p {
        display: inline-block;
        font-weight: 500;
        line-height: 1.4
      }

      .cta:hover .cta-arrow,
      .cta:focus .cta-arrow,
      .link-block:hover .cta-arrow,
      .link-block:focus .cta-arrow {
        transform: translateX(.25em)
      }

      .cta:hover .cta-arrow.cta-arrow-inverse,
      .cta:focus .cta-arrow.cta-arrow-inverse {
        transform: translateX(-.25em)
      }

      .clamp {
        overflow: hidden;
        display: -webkit-box;
        -webkit-box-orient: vertical;
        -webkit-line-clamp: 3
      }

      .clamp.clamp2 {
        -webkit-line-clamp: 2
      }

      .item::after {
        content: "";
        background: radial-gradient(circle at 50%, #d8a76e, #237F99);
        display: block;
        position: absolute;
        bottom: 0;
        width: 100%;
        height: 1px
      }

      .link-block .cta {
        transition: opacity 500ms
      }

      .link-block:hover .cta,
      .link-block:focus .cta,
      .link-external:hover~a .cta,
      .link-external:focus~a .cta {
        opacity: 60%
      }

      .rt blockquote+h6 {
        color: #fff;
        position: relative;
        bottom: 7em;
        margin-bottom: -1em
      }

      .rt blockquote+h6 em {
        font-weight: 300;
        color: #D8A76E;
        margin-left: 1.4em
      }

      .rt figure,
      .rt .w-embed {
        margin: 3.75em 0
      }

      .rt figure:last-child,
      .rt .w-embed:last-child {
        margin-bottom: 0
      }

      .rt figure,
      .rt .w-embed {
        border-radius: .938em;
        -webkit-mask-image: -webkit-radial-gradient(white, black)
      }

      .rt figure.w-richtext-figure-type-image,
      .article img {
        border-radius: .625em
      }

      .s-privacy .rt a {
        color: #b1c3e8
      }

      .bg {
        transform: scale3d(1, 1, 1);
        transform-style: preserve-3d
      }

      .link-qa:hover .bg,
      .link-qa:focus .bg,
      .link-external:hover~.link-qa .bg,
      .link-external:focus~.link-qa .bg {
        transform: scale3d(1.025, 1.025, 1)
      }

      .link-qa:hover .btn,
      .link-qa:focus .btn {
        filter: brightness(120%);
        -webkit-filter: brightness(120%)
      }

      .featured .w-dyn-list,
      .featured .w-dyn-items,
      .featured .w-dyn-item,
      .featured .link-qa {
        height: 100%
      }

      .w-file-upload-default,
      .w-file-upload-uploading {
        height: 3.176470588235294em;
        display: block
      }

      .w-file-upload-file-name {
        font-size: inherit
      }

      @media screen and (max-width:479px) {
        .link-footer:not(:first-child) {
          margin-right: 0
        }

        .rt blockquote+h6,
        .rt blockquote+h6 em {
          margin-left: 0 !important
        }

        .rt blockquote+h6 {
          bottom: 6em !important;
          margin-bottom: -3em !important
        }

        .rt blockquote+h6 em {
          display: block
        }

        .link-qa img {
          transform: none !important
        }

        .link-qa {
          color: #237F99 !important
        }

        .s1-home,
        .s1-about,
        .s1-pipeline,
        .s1-science {
          max-height: none !important
        }

        .s1.s1-team {
          max-height: -webkit-fill-available !important
        }

        .link-contact .btn {
          width: 100%;
          text-align: center
        }

        .s1 {
          max-height: -webkit-fill-available
        }
      }

      @media screen and (min-width:480px) {
        .abs-qa {
          background-color: transparent !important
        }
      }

      @media screen and (max-width:767px) {

        .rt figure,
        .rt .w-embed {
          margin: 2.5em 0
        }

        .link-team h3 {
          font-size: 1.5em
        }

        .rt h2:first-child {
          margin-top: 1.5em
        }

        .card-modal {
          min-height: -webkit-fill-available
        }
      }

      @media screen and (min-width:768px) and (max-width:991px) {
        .s1 {
          max-height: -webkit-fill-available
        }
      }

      @media screen and (max-width:991px) {
        .rt blockquote+h6 {
          margin-left: 60px;
          bottom: 4.5em;
          margin-bottom: -1.5em
        }

        .rt blockquote+h6 em {
          margin-left: 1em
        }

        .item .pc.pc-4 {
          margin-top: .2em
        }
      }

      @media screen and (min-width:992px) {
        .s.s1:not(.s1-team) {
          height: calc(100vh + 1vw)
        }

        .s.s1.s1-about {
          height: calc(100vh + 5vw)
        }

        .s.s1.s1-science {
          height: calc(100vh + 2vw)
        }

        .row-team:nth-child(3) .grid {
          -ms-grid-columns: 1fr 1fr 1fr 1fr;
          grid-template-columns: 1fr 1fr 1fr 1fr
        }

        .row-team:nth-child(3) h3 {
          font-size: 1.25em;
          font-weight: 500
        }

        .menu {
          background: none !important
        }

        .grid-qa-2 .w-dyn-item:first-child {
          display: none
        }
      }

      @media screen and (min-width:992px) and (max-width:1180px) {
        .rt blockquote+h6 {
          left: 2.75em
        }

        .rt blockquote {
          margin-left: -1.5em;
          margin-right: -1.5em
        }

        .c1-contact {
          padding-top: 30px
        }

        .left-110 {
          padding-left: 0
        }

        .right-110 {
          padding-right: 0
        }

        .c2-contact .left-110 {
          padding-left: 3em
        }

        .c2-contact .right-110 {
          padding-right: 3em
        }

        .grid.grid-pipeline {
          font-size: 1.2vw
        }
      }

      @media screen and (min-width:992px) and (max-width:1360px) {
        body {
          font-size: 14px
        }
      }

      @media screen and (min-width:992px) and (max-width:1360px) {
        .p-17.p-pipeline-phase.categ {
          font-size: 16px
        }
      }

      @media screen and (min-width:1792px) {
        .grid.grid-gap-143 {
          grid-column-gap: 8.9375em
        }

        .left-110 {
          padding-left: 6.875em
        }

        .right-110 {
          padding-right: 6.875em
        }
      }

      @media screen and (min-width:2200px) {
        body {
          font-size: 18px
        }
      }

      .custom-width {
        width: 75%;
      }


      .table_visiting_hours {
        width: 80%;
        margin: 0 auto;
        padding: 20px;
        text-align: center;
        background: rgba(255, 255, 255, 0.1);
        border-radius: 10px;
        box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
        backdrop-filter: blur(5px);
        -webkit-backdrop-filter: blur(5px);
      }
    </style>
  </div>


  <!--MEDIA QUERIES FOR LOGO-->
  <style>
    @media (max-width:768px) {
      .logo {
        margin-left: -140px !important;
        height: 0px;
      }

    }
  </style>



  <div class="tr" data-ix="tr"></div>



  <div data-collapse="medium" data-animation="default" data-duration="800" data-easing="ease-out-expo"
    data-easing2="ease-out-expo" data-w-id="710a0016-3feb-10ed-f3e7-f258a52639cb" role="banner" class="s nav w-nav">




    <nav class="c">
      <div class="logo"><a href="../../index.html" aria-label="home" class="w-inline-block">
          <div>

            <svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="119px" height="85px"
              style="shape-rendering:geometricPrecision; text-rendering:geometricPrecision; image-rendering:optimizeQuality; fill-rule:evenodd; clip-rule:evenodd"
              xmlns:xlink="http://www.w3.org/1999/xlink">
              <g>
                <path style="opacity:0.285" fill="#325c2b"
                  d="M 19.5,14.5 C 20.2891,14.7828 20.9558,15.2828 21.5,16C 19.3996,24.4269 21.5663,31.5936 28,37.5C 28.6877,38.3317 28.521,38.9984 27.5,39.5C 16.4229,33.7115 13.7562,25.3782 19.5,14.5 Z" />
              </g>
              <g>
                <path style="opacity:0.908" fill="#335a2c"
                  d="M 51.5,35.5 C 55.4589,34.5778 57.7922,36.0778 58.5,40C 56.78,43.4649 54.1133,44.2982 50.5,42.5C 49.073,39.8845 49.4063,37.5511 51.5,35.5 Z" />
              </g>
              <g>
                <path style="opacity:0.907" fill="#335c2c"
                  d="M 59.5,35.5 C 61.8333,35.5 64.1667,35.5 66.5,35.5C 66.1796,38.8984 66.513,42.2317 67.5,45.5C 72.0589,43.8889 76.2256,44.5556 80,47.5C 85.2396,66.0948 78.4063,73.0948 59.5,68.5C 59.5,57.5 59.5,46.5 59.5,35.5 Z M 68.5,48.5 C 71.368,48.085 73.5347,49.085 75,51.5C 75.9777,55.9002 75.4777,60.0669 73.5,64C 67.6575,65.1706 65.3242,62.6706 66.5,56.5C 66.6205,53.5589 67.2872,50.8922 68.5,48.5 Z" />
              </g>
              <g>
                <path style="opacity:0.275" fill="#325d2c"
                  d="M 36.5,6.5 C 46.9026,14.8006 48.736,24.8006 42,36.5C 35.9588,42.7089 29.7921,48.7089 23.5,54.5C 25.5,55.1667 27.5,55.8333 29.5,56.5C 26.5,57.1667 23.5,57.8333 20.5,58.5C 19.9598,59.7481 19.2931,60.9148 18.5,62C 32.8547,63.3077 32.5213,64.141 17.5,64.5C 16.7588,68.7947 16.5921,73.128 17,77.5C 12.3788,69.2313 12.3788,60.898 17,52.5C 22.6771,45.6567 28.8438,39.3233 35.5,33.5C 33.1667,32.8333 30.8333,32.1667 28.5,31.5C 31.8333,30.8333 35.1667,30.1667 38.5,29.5C 39.2458,28.2638 39.5792,26.9305 39.5,25.5C 36.1016,25.8204 32.7683,25.487 29.5,24.5C 33.1073,23.5108 36.774,23.1774 40.5,23.5C 40.5,22.1667 40.5,20.8333 40.5,19.5C 24.5137,18.9959 24.5137,18.1626 40.5,17C 38.7396,13.6488 37.4063,10.1488 36.5,6.5 Z" />
              </g>
              <g>
                <path style="opacity:0.897" fill="#325c2d"
                  d="M 50.5,44.5 C 52.8333,44.5 55.1667,44.5 57.5,44.5C 57.5,52.5 57.5,60.5 57.5,68.5C 55.1667,68.5 52.8333,68.5 50.5,68.5C 50.5,60.5 50.5,52.5 50.5,44.5 Z" />
              </g>
              <g>
                <path style="opacity:0.889" fill="#325c2c"
                  d="M 87.5,44.5 C 92.2142,44.3531 96.8809,44.6864 101.5,45.5C 102.052,47.3307 102.052,48.9974 101.5,50.5C 97.4001,48.0246 93.4001,48.0246 89.5,50.5C 93.6423,52.39 97.809,54.39 102,56.5C 104.429,62.5457 102.596,66.7123 96.5,69C 92.141,69.5343 87.8076,69.3676 83.5,68.5C 83.5792,67.0695 83.2458,65.7362 82.5,64.5C 83,64.1667 83.5,63.8333 84,63.5C 87.4815,65.5917 90.9815,65.7584 94.5,64C 95.7472,62.5064 95.7472,61.0064 94.5,59.5C 91.5499,58.5123 88.5499,57.679 85.5,57C 82.6508,54.4406 81.8175,51.2739 83,47.5C 84.4087,46.2189 85.9087,45.2189 87.5,44.5 Z" />
              </g>
              <g>
                <path style="opacity:0.851" fill="#325b2c"
                  d="M 34.5,45.5 C 36.679,45.84 38.5123,46.84 40,48.5C 43.6939,54.5545 46.8606,60.8878 49.5,67.5C 46.9135,68.48 44.2469,68.8134 41.5,68.5C 40.9659,60.2348 38.6325,52.5681 34.5,45.5 Z" />
              </g>
              <g>
                <path style="opacity:0.395" fill="#237F99"
                  d="M 72.5,71.5 C 74.9509,73.478 77.6176,73.8114 80.5,72.5C 81.3081,72.8076 81.9747,73.3076 82.5,74C 81.3595,74.7412 81.3595,75.5745 82.5,76.5C 78.8352,77.2329 74.8352,77.5662 70.5,77.5C 70.67,76.1779 70.3366,75.0113 69.5,74C 70.2529,72.513 71.0862,72.3463 72,73.5C 72.4643,72.9056 72.631,72.2389 72.5,71.5 Z" />
              </g>
              <g>
                <path style="opacity:0.459" fill="#325d2c"
                  d="M 99.5,71.5 C 100.5,71.5 101.5,71.5 102.5,71.5C 102.5,73.5 102.5,75.5 102.5,77.5C 99.5525,77.7771 96.7191,77.4438 94,76.5C 93.1282,72.8651 94.2949,71.8651 97.5,73.5C 98.7133,73.2528 99.38,72.5862 99.5,71.5 Z" />
              </g>
              <g>
                <path style="opacity:0.507" fill="#325c2c"
                  d="M 49.5,71.5 C 53.0324,71.3789 55.8658,72.7122 58,75.5C 58.9356,73.2915 60.6023,71.9582 63,71.5C 64.4007,73.1922 66.2341,73.8588 68.5,73.5C 68.5,74.8333 68.5,76.1667 68.5,77.5C 67.1667,77.5 65.8333,77.5 64.5,77.5C 64.1667,73.5 63.8333,73.5 63.5,77.5C 60.319,77.2577 57.319,77.2577 54.5,77.5C 53.3756,72.2792 52.5423,72.2792 52,77.5C 51.4153,75.3386 50.5819,73.3386 49.5,71.5 Z" />
              </g>
              <g>
                <path style="opacity:0.393" fill="#335d2b"
                  d="M 83.5,73.5 C 86.5,73.5 89.5,73.5 92.5,73.5C 92.5,74.8333 92.5,76.1667 92.5,77.5C 89.863,76.7678 87.363,76.7678 85,77.5C 83.8118,76.4366 83.3118,75.1033 83.5,73.5 Z" />
              </g>
            </svg>



          </div>
        </a></div>


      <nav role="navigation" id="menu" class="menu w-nav-menu">
        <div class="menu-inner">
          <div class="no-tb"></div>
          <div class="center"><a href="../../about.html" tabindex="0" class="navlink">About</a><a
              href="../../index-dna-test.html" tabindex="0" class="navlink">DNATest</a><a href="training.html"
              class="navlink">Training</a><a href="../../solutions.html" class="navlink">Solutions</a><a
              href="#" class="navlink">Careers</a><a href="#"
              class="navlink">News &amp; Publications</a><a href="#" class="navlink">Cart</a>
          </div>
          <div class="right"><a role="button" href="../../contact.html" target="_blank" aria-current="page"
              class="btn btn-sm w-button w--current">Contact Us</a></div>
        </div>
      </nav>
      <div class="menu-btn w-nav-button">
        <div class="menu-bar menu-bar-top"></div>
        <div class="menu-bar menu-bar-mid"></div>
        <div class="menu-bar menu-bar-bot"></div>
      </div>
    </nav>
  </div>









  <div id="content" role="main" class="page-section--main">
    <div data-elementor-type="single-post" data-elementor-id="83318"
      class="elementor elementor-83318 elementor-location-single post-15822 post type-post status-publish format-standard has-post-thumbnail hentry category-employee-training tag-training-fundamentals tag-types-of-training"
      data-elementor-post-type="elementor_library">
      <section
        class="elementor-section elementor-top-section elementor-element elementor-element-403c6be elementor-section-boxed elementor-section-height-default elementor-section-height-default"
        data-id="403c6be" data-element_type="section"
        data-settings="{&quot;background_background&quot;:&quot;classic&quot;}">
        <div class="elementor-container elementor-column-gap-default">
          <div
            class="elementor-column elementor-col-33 elementor-top-column elementor-element elementor-element-1cefc18 elementor-hidden-tablet"
            data-id="1cefc18" data-element_type="column">
            <div class="elementor-widget-wrap">
            </div>
          </div>
          <div
            class="elementor-column elementor-col-33 elementor-top-column elementor-element elementor-element-435be91"
            data-id="435be91" data-element_type="column">
            <div class="elementor-widget-wrap elementor-element-populated">
              <div
                class="elementor-element elementor-element-2d0ac7f7 elementor-align-left elementor-widget elementor-widget-post-info"
                data-id="2d0ac7f7" data-element_type="widget" data-widget_type="post-info.default">
                <div class="elementor-widget-container">
                  <ul class="elementor-inline-items elementor-icon-list-items elementor-post-info">
                    <li class="elementor-icon-list-item elementor-repeater-item-f9b36c3 elementor-inline-item"
                      itemprop="about">
                      <span
                        class="elementor-icon-list-text elementor-post-info__item elementor-post-info__item--type-terms">
                        <span class="elementor-post-info__terms-list">
                          <a href="https://whatfix.com/blog/category/employee-training/"
                            class="elementor-post-info__terms-list-item">Employee Training</a> </span>
                      </span>
                    </li>
                  </ul>
                </div>
              </div>
              <div
                class="elementor-element elementor-element-50101e26 elementor-widget elementor-widget-theme-post-title elementor-page-title elementor-widget-heading"
                data-id="50101e26" data-element_type="widget" data-widget_type="theme-post-title.default">
                <div class="elementor-widget-container">
                  <h1 class="elementor-heading-title elementor-size-default">BIOINFORMATICS TRAINING</h1>
                </div>
              </div>
              <div
                class="elementor-element elementor-element-79f93b7a elementor-align-left elementor-widget elementor-widget-post-info"
                data-id="79f93b7a" data-element_type="widget" data-widget_type="post-info.default">
                <div class="elementor-widget-container">
                  <ul class="elementor-inline-items elementor-icon-list-items elementor-post-info">
                    <li class="elementor-icon-list-item elementor-repeater-item-5b276ce elementor-inline-item"
                      itemprop="datePublished">
                      <span
                        class="elementor-icon-list-text elementor-post-info__item elementor-post-info__item--type-date">
                        <span class="elementor-post-info__item-prefix">Published:</span>
                        <time>January 24th, 2025</time> </span>
                    </li>
                    <li class="elementor-icon-list-item elementor-repeater-item-67ffae1 elementor-inline-item">
                      <span
                        class="elementor-icon-list-text elementor-post-info__item elementor-post-info__item--type-custom">
                      </span>
                    </li>
                  </ul>
                </div>
              </div>
              <div
                class="elementor-element elementor-element-adb67ee elementor-align-left elementor-widget elementor-widget-post-info"
                data-id="adb67ee" data-element_type="widget" data-widget_type="post-info.default">
                <div class="elementor-widget-container">
                  <ul class="elementor-inline-items elementor-icon-list-items elementor-post-info">
                    <li class="elementor-icon-list-item elementor-repeater-item-61f0c4f elementor-inline-item"
                      itemprop="author">
                      <span
                        class="elementor-icon-list-text elementor-post-info__item elementor-post-info__item--type-author">
                        <span class="elementor-post-info__item-prefix">By</span>
                        Asenass Developers </span>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
          <div
            class="elementor-column elementor-col-33 elementor-top-column elementor-element elementor-element-4666da8b elementor-hidden-tablet"
            data-id="4666da8b" data-element_type="column">
            <div class="elementor-widget-wrap">
            </div>
          </div>
        </div>
      </section>
      <section
        class="elementor-section elementor-top-section elementor-element elementor-element-7a1a822d elementor-section-boxed elementor-section-height-default elementor-section-height-default"
        data-id="7a1a822d" data-element_type="section">
        <div class="elementor-container elementor-column-gap-default">
          <div
            class="elementor-column elementor-col-33 elementor-top-column elementor-element elementor-element-6b8dfd96 elementor-hidden-tablet elementor-hidden-phone"
            data-id="6b8dfd96" data-element_type="column"
            data-settings="{&quot;background_background&quot;:&quot;classic&quot;}">
            <div class="elementor-widget-wrap elementor-element-populated">
              <div
                class="elementor-element elementor-element-607203f8 elementor-toc--minimized-on-tablet elementor-widget elementor-widget-table-of-contents"
                data-id="607203f8" data-element_type="widget"
                data-settings="{&quot;exclude_headings_by_selector&quot;:&quot;.featuredno&quot;,&quot;sticky&quot;:&quot;top&quot;,&quot;sticky_parent&quot;:&quot;yes&quot;,&quot;sticky_offset&quot;:90,&quot;headings_by_tags&quot;:[&quot;h2&quot;],&quot;marker_view&quot;:&quot;bullets&quot;,&quot;icon&quot;:{&quot;value&quot;:&quot;fas fa-circle&quot;,&quot;library&quot;:&quot;fa-solid&quot;,&quot;rendered_tag&quot;:&quot;&lt;svg class=\&quot;e-font-icon-svg e-fas-circle\&quot; viewBox=\&quot;0 0 512 512\&quot; xmlns=\&quot;http:\/\/www.w3.org\/2000\/svg\&quot;&gt;&lt;path d=\&quot;M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8z\&quot;&gt;&lt;\/path&gt;&lt;\/svg&gt;&quot;},&quot;minimize_box&quot;:&quot;yes&quot;,&quot;minimized_on&quot;:&quot;tablet&quot;,&quot;hierarchical_view&quot;:&quot;yes&quot;,&quot;min_height&quot;:{&quot;unit&quot;:&quot;px&quot;,&quot;size&quot;:&quot;&quot;,&quot;sizes&quot;:[]},&quot;min_height_tablet&quot;:{&quot;unit&quot;:&quot;px&quot;,&quot;size&quot;:&quot;&quot;,&quot;sizes&quot;:[]},&quot;min_height_mobile&quot;:{&quot;unit&quot;:&quot;px&quot;,&quot;size&quot;:&quot;&quot;,&quot;sizes&quot;:[]},&quot;sticky_on&quot;:[&quot;desktop&quot;,&quot;tablet&quot;,&quot;mobile&quot;],&quot;sticky_effects_offset&quot;:0,&quot;sticky_anchor_link_offset&quot;:0}"
                data-widget_type="table-of-contents.default">




                <!-- THE TABLE OF CONTENTS GOES IN HERE. -->
                







                <div class="elementor-widget-container">
                  <div class="elementor-toc__header">
                    <div class="elementor-toc__header-title">Table of Contents</div>
                    <div class="elementor-toc__toggle-button elementor-toc__toggle-button--expand" role="button" tabindex="0"
                      aria-controls="elementor-toc__body" aria-expanded="true" aria-label="Open table of contents">
                      <svg aria-hidden="true" class="e-font-icon-svg e-fas-chevron-down" viewBox="0 0 448 512"
                        xmlns="http://www.w3.org/2000/svg">
                        <path
                          d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z">
                        </path>
                      </svg>
                    </div>
                    <div class="elementor-toc__toggle-button elementor-toc__toggle-button--collapse" role="button" tabindex="0"
                      aria-controls="elementor-toc__body" aria-expanded="true" aria-label="Close table of contents">
                      <svg aria-hidden="true" class="e-font-icon-svg e-fas-chevron-up" viewBox="0 0 448 512"
                        xmlns="http://www.w3.org/2000/svg">
                        <path
                          d="M240.971 130.524l194.343 194.343c9.373 9.373 9.373 24.569 0 33.941l-22.667 22.667c-9.357 9.357-24.522 9.375-33.901.04L224 227.495 69.255 381.516c-9.379 9.335-24.544 9.317-33.901-.04l-22.667-22.667c-9.373-9.373-9.373-24.569 0-33.941L207.03 130.525c9.372-9.373 24.568-9.373 33.941-.001z">
                        </path>
                      </svg>
                    </div>
                  </div>
                  <div id="elementor-toc__body" class="elementor-toc__body">
                    <ul id="table-of-contents-list"></ul>
                  </div>
                </div>
                
                <script>
                  document.addEventListener("DOMContentLoaded", function () {
                    // Select all h2 elements on the page
                    const headings = document.querySelectorAll("h2");
                    const tocList = document.getElementById("table-of-contents-list");
                
                    // Loop through each h2 and create an entry in the TOC
                    headings.forEach((heading, index) => {
                      // Ensure each heading has an ID for linking
                      if (!heading.id) {
                        heading.id = `toc-heading-${index + 1}`;
                      }
                
                      // Create a new list item with a link to the heading
                      const listItem = document.createElement("li");
                      const link = document.createElement("a");
                      link.href = `#${heading.id}`;
                      link.textContent = heading.textContent;
                      listItem.appendChild(link);
                
                      // Append the list item to the TOC
                      tocList.appendChild(listItem);
                    });
                  });
                </script>
                
                <style>
                  .elementor-toc__body ul {
                    list-style: none;
                    padding: 0;
                    margin: 0;
                  }
                
                  .elementor-toc__body li {
                    margin: 0.5em 0;
                  }
                
                  .elementor-toc__body a {
                    text-decoration: none;
                    color: #0073e6;
                    font-weight: bold;
                  }
                
                  .elementor-toc__body a:hover {
                    text-decoration: underline;
                  }
                </style>



              </div>
            </div>
          </div>
          <div
            class="elementor-column elementor-col-33 elementor-top-column elementor-element elementor-element-3a8f3f5b"
            data-id="3a8f3f5b" data-element_type="column">
            <div class="elementor-widget-wrap elementor-element-populated">
              <div
                class="elementor-element elementor-element-13ef5594 elementor-widget elementor-widget-theme-post-content"
                data-id="13ef5594" data-element_type="widget" data-widget_type="theme-post-content.default">



                <!-- THE BEGINNING OF THE BODY -->
                <div class="elementor-widget-container">
                  <!-- <p>The importance of employee training and continued skill development can&#8217;t be understated. It
                    has an impact on direct and indirect business outcomes, including productivity, revenue, turnover,
                    and more.</p> -->



                  <!-- <p>According to <a
                      href="https://hbr.org/2022/04/3-ways-to-boost-retention-through-professional-development">Deloitte</a>,
                    organizations with a strong learning culture are 92% more likely to develop new product innovations
                    or novel process improvements<span style="box-sizing: border-box; margin: 0px; padding: 0px;"> and
                      are 52% more productive overall. However, <a
                        href="https://www.forbes.com/sites/forbesagencycouncil/2023/08/15/why-training-is-an-investment-not-an-expense/"
                        target="_blank" rel="noopener">59%</a> of employees claim they received no workplace training
                      and that the skills required to do their jobs</span> effectively were self-taught.</p>
                  <p>This highlights a curious trend of employees relying on self-taught skills despite organizations
                    acknowledging the importance of workplace training for professional growth, upskilling, and team
                    improvement. Companies view employee training as an expense rather than an investment. Many
                    organizations must also maintain current output and productivity levels before focusing on learning
                    and development opportunities.</p>
                  <p>Employee training programs and development opportunities are essential features of successful
                    organizations. They enable them to upskill team members and drive productivity levels. These
                    programs are not a means to fill knowledge gaps; they&#8217;re strategic investments in people,
                    fostering a culture of continuous improvement and adaptability.</p>
                  <p>From onboarding new hires to upskilling workers for emerging responsibilities, employee training
                    programs are designed to align individual growth with organizational goals, futureproof
                    organizations, and enable them to remain competitive, innovate, and achieve business outcomes.</p>
                  <div class="featuredno"> -->





                  <!-- THE LIST OF COURSE TITLES: -->
                  <h2>COURSE TITLES</h2>
                  <ol>
                    <li style="font-weight: 400;" aria-level="1"><a href="">Bioinformatics 1</a></li>
                    <li style="font-weight: 400;" aria-level="1"><a href="">NextGen Sequencing (NGS) Data Analysis</a>
                    </li>
                    <li style="font-weight: 400;" aria-level="1"><a href="">Metagenomicsv</a></li>
                    <li style="font-weight: 400;" aria-level="1"><a href="">Metabolomics</a>
                    </li>
                    <li style="font-weight: 400;" aria-level="1"><a href="">Computer-Aided Drug Design</a></li>
                    <li style="font-weight: 400;" aria-level="1"><a href="">Designing Point of Care Diagnostics using
                        the CRISPR-Cas System</a>
                    </li>
                    <li style="font-weight: 400;" aria-level="1"><a href="">Genome Editing using CRISPR-Cas</a></li>
                    <li style="font-weight: 400;" aria-level="1"><a href="">PCR Boot Camp</a></li>
                    <li style="font-weight: 400;" aria-level="1"><a href="">Python Programming for Bio scientists</a>
                    </li>
                    <li style="font-weight: 400;" aria-level="1"><a href="">Techniques in Nanopore DNA/RNA Sequencing
                        and Data Analysis</a></li>
                  </ol>
                </div>




                <h2>What Is an Training Program?</h2>
                <p>A training program is a structured plan designed to develop specific skills, knowledge, or
                  competencies in individuals, often to enhance performance in a particular area or role. It typically
                  includes activities like workshops, courses, or hands-on exercises.</p>

                <br><br><br>


                <!-- THE COURSES: -->
                <section>
                  <style>
                    .myh4 {
                      color: #081B42;
                      font-family: cursive;
                      font-size: 50px;
                    }
                  </style>

                  <!-- HEADER FOR THE COURSE DESCRIPTIONS -->
                  <h2><strong>COURSE DESCRIPTION</strong></h2>


                  <!-- THE COURSER GO IN HERE: -->
                  <ol>
                    <li>
                      <h2>Bioinformatics 1</h2>
                      <h4 class="myh4">About The Course</h4>
                      <p>In this course, the student is introduced to the concepts of Bioinformatics. Hands-on sessions
                        will familiarize students with the details and use of common tools and resources. The objective
                        is to enhance the researchers’ capacity to conduct world class scientific research and boost
                        their institution’s profile as a leading diagnostic and scientific center.</p>
                      <p>
                        At the end of the training, the participant will be able to:
                      </p>
                      <ul>
                        <li>Learn the science, methods, and applications of bioinformatics</li>
                        <li>Analyze biological datasets and discover patterns and trends</li>
                        <li>Discover, design and develop drugs in silico</li>
                        <li>Discover and identify biomarkers</li>
                        <li>Design CRISPR-cas guide RNAs and PCR and RPA primers and probes and develop point of care
                          diagnostic kits</li>
                        <li>Appreciate the role of metabolomics and proteomics in scientific research</li>
                        <li>Perform common molecular laboratory techniques</li>
                        <li>Identify genetic diseases, resistance genes, and gene variants using raw sequencing data and
                          optimize precision medicine</li>

                      </ul>

                      <h4 class="myh4">Resources</h4>
                      <p>Using the most powerful software, you will gain the latest skills in bioinformatics analysis.
                      </p>
                      <ul>
                        <li>
                          <i>Web Resources - </i>
                          <ul>
                            <li><a href="https://www.ncbi.nlm.nih.gov/guide/all/">NCBI Tools</a></li>
                            <li><a href="https://www.expasy.org/">EXPASY Tools</a></li>
                            <li><a href="https://www.rcsb.org/">RCSB PDB</a></li>
                            <li><a href="https://www.cathdb.info/">CATH</a></li>
                            <li><a href="https://scop2.mrc-lmb.cam.ac.uk/">SCOP</a></li>
                            <li><a href="https://string-db.org/">STRING</a></li>
                            <li><a href="https://www.genome.gov/genetics-glossary/hapmap">HapMap</a></li>
                            <li><a href="">SKAT-O</a></li>
                            <li><a
                                href="https://gatk.broadinstitute.org/hc/en-us/articles/360035890811-Resource-bundle">Broad
                                Institute</a></li>
                            <li><a href="https://www.ncbi.nlm.nih.gov/clinvar/">ClinVar</a></li>
                            <li><a href="https://metlin.scripps.edu/landing_page.php?pgcontent=mainPage">METLIN</a></li>
                            <li><a href="https://hmdb.ca/">HMDB</a></li>
                            <li><a href="https://www.metaboanalyst.ca/">MetaboAnalyst</a></li>
                            <li><a href="https://www.ebi.ac.uk/metagenomics/">MGnify</a></li>
                            <li><a href="https://www.ncbi.nlm.nih.gov/geo/geo2r/">GEO2R</a></li>
                            <li><a href="https://zinc.docking.org/">ZINC</a></li>
                            <li><a href="https://pubchem.ncbi.nlm.nih.gov/">PubChem</a></li>
                            <li><a href="">ADME-Tox</a></li>
                            <li><a href="https://webs.iiitd.edu.in/raghava/toxipred/">ToxiPred</a></li>
                            <li><a href="https://alphafold.ebi.ac.uk/">Alpha Fold</a></li>
                          </ul>
                        <li>
                          <i>Software -</i>
                          <ul>
                            <li>Perl Distribution</li>
                            <li>SRA Toolkits</li>
                            <li>BBMAP</li>
                            <li>Fastqc</li>
                            <li>BWA</li>
                            <li>GATK</li>
                            <li>Samtools</li>
                            <li>Picard</li>
                            <li><a href="https://www.cog-genomics.org/plink/">PLINK</a></li>
                            <li>IGV</li>
                            <li>MEGA</li>
                            <li>Mr. Bayes</li>
                            <li>FigTree</li>
                            <li>LAMP/WAMP</li>
                            <li>R</li>
                            <li>MEGAN</li>
                            <li>AutoDock Tools</li>
                            <li>Tools</li>
                            <li>Chimera</li>
                            <li>Marvin Sketch</li>
                            <li>Raccoon</li>
                            <li>PyRx</li>
                            <li>Ligand Scout</li>
                          </ul>
                        <li>
                          <i>HPC - </i>
                          <ul>
                            <li>KENCLUST</li>
                          </ul>
                        </li>
                        <li>
                          <i>Datasets</i>
                          <ul>
                            <li>SRA Datasets</li>
                            <li>1,000 Genomes Project</li>
                            <li>The Cancer Genome Atlas (TCGA)</li>
                          </ul>
                        </li>
                    </li>

                    </li>
                    </ul>
                    <h4 class="myh4">Training program</h4>
                    <p>This is a 12-week intensive training program covering the following topics:</p>
                    <ul>
                      <li>Introduction to Bioinformatics</li>
                      <li>Introduction to Linux OS and Cluster Computing</li>
                      <li>Python Programming for Biologists</li>
                      <li>Fundamentals of Bioinformatics</li>
                      <li>Proteomics.</li>
                      <li>NGS Data Analysis.</li>
                      <li>Phylogenetics</li>
                      <li>Biological Databases, Design and Deployment</li>
                      <li>Transcriptomics and Microarray Analysis</li>
                      <li>Metagenomics</li>
                      <li>Metabolomics and Systems Biology</li>
                      <li>Computer-Aided Drug Design</li>



                    </ul>

                    <h4 class="myh4">Prerequisite and fees</h4>
                    <p>The course is ideal for students and researchers working in the pharma, biotech, and medical
                      industries. The following terms and conditions apply.</p>


                    <ul>
                      <li>The fee is Kshs. 100,000/ for East African residents. Non-East African students pay 1,000 USD.
                      </li>
                      <li>Fees covers tuition, training material, and certificate only.</li>
                      <li>
                        Fees are payable strictly in advance (at least 4 working days before the starting date.
                      </li>
                      <li>Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                        least 2 weeks before the start) will be offered a 5% discount</li>
                      <li>There will be a 10% administrative charge for cancellations received in writing up to 20
                        working days before the start of the course. No refunds will be made for cancellations received
                        within 20 working days of the course start date or for the inability to attend the course for
                        whatever reasons. Substitutions may, of course, be made at any time, providing you inform us in
                        writing. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                        other reason that might lead to a delegate not being able to attend the course.</li>
                      <li>Fee is payable by direct depsoit, bankers cheque or MPESA to: Bioinformatics Institute of
                        Kenya Ltd. A/C No. 0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya.
                        The Paybill number for payments via Mpesa is 400222 and the account number is 1679821#your name.
                        Contact us for online payments.</li>
                    </ul>

                    <h4 class="myh4">Enrollment dates</h4>
                    <ul>
                      <li>January - March</li>
                      <li>April - June</li>
                      <li>July - September</li>
                      <li>October-December</li>
                    </ul>
                    </li>






                    <!-- COURSE NUMBER TWO -->
                    <!-- NGS Data Analysis Course -->
                    <li>
                      <h2>NGS Data Analysis Course</h2>

                      <!-- ABOUT THE COURSE -->
                      <h4 class="myh4">About The Course</h4>
                      <p>NGS Data Analysis Course is a hands-on training designed to equip participants with knowledge
                        and skills on how to generate, manipulate, and analyze whole genome, whole exome, and targeted
                        NGS data. This course equips participants with skills on how to discover germline SNPs, indels,
                        SVs, and CNVs; somatic SNVs, SVs, and indels, and RNASeq SNPs and indels. The objective is to
                        enhance the researchers’ capacity to conduct world class scientific research and boost their
                        institution’s profile as a leading diagnostic and scientific center.
                      </p>

                      <!-- AT THE END OF THE TRAINING, THE PARTICIPANT SHOULD BE ABLE TO: -->
                      <p>
                        At the end of the training, the participant will be able to:
                      </p>
                      <!-- LIST OF DELIVERABLES PARTICIPANT SHOULD HAVE AT THE END OF THE TRAINING PROGRAMME. -->
                      <ul>
                        <li>Learn the science, methods and applications of Next Generation Sequencing (NGS)</li>
                        <li>Discover single nucleotide polymorphisms (SNPs), indels, copy number variants (CNVs),
                          structural variants (SVs), and single nucleotide variants (SNVs) from raw sequences</li>
                        <li>Identify variants of clinical significance</li>
                        <li>Optimize therapy and diagnosis and improve clinical decisioning based on the discovered
                          variants</li>
                      </ul>

                      <!-- RESOURCES... -->
                      <h4 class="myh4">Resources</h4>
                      <p>Using the most powerful software, you will gain the latest skills in NGS data analysis.
                      </p>
                      <ul>
                        <li>
                          <i>Web Resources - </i>
                          <ul>
                            <li><a href="https://www.ncbi.nlm.nih.gov/guide/all/">NCBI Tools</a></li>
                            <li><a href="https://www.genome.gov/genetics-glossary/hapmap">HapMap</a></li>
                            <li><a href="https://www.cog-genomics.org/plink/">PLINK</a></li>
                            <li><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3415556/">SKAT-O</a></li>
                            <li><a
                                href="https://gatk.broadinstitute.org/hc/en-us/articles/360035890811-Resource-bundle">
                                Broad Institute Resource Bundle</a></li>
                          </ul>
                        <li>

                          <i>Software -</i>
                          <ul>
                            <li>SRA Toolkits</li>
                            <li>BBMAP</li>
                            <li>Fastqc</li>
                            <li>BWA</li>
                            <li>GATK</li>
                            <li>Samtools</li>
                            <li>Picard</li>
                            <li>IGV</li>
                          </ul>


                        <li>
                          <i>High performance computing cluster - </i>
                          <ul>
                            <li>KENCLUST</li>
                          </ul>
                        </li>


                        <li>
                          <i>Datasets</i>
                          <ul>
                            <li>SRA Datasets</li>
                            <li>1,000 Genomes Project</li>
                            <li>The Cancer Genome Atlas (TCGA)</li>
                          </ul>
                        </li>

                    </li>

                    </li>
                    </ul>


                    <!-- TRAINING PROGRAMME: -->
                    <h4 class="myh4">Training program</h4>
                    <p>This is an intensive training program covering the following topics:</p>
                    <ul>
                      <li>NestGen Sequencing (NGS) principles, methods and technologies</li>
                      <li>NGS in Clinical Practice: Applications, Challenges and Promises</li>
                      <li>Introduction to the Linux OS and basic commands</li>
                      <li>SSH, Telnet, FTP, WinSCP, Putty, XMING, port forwarding and remote access</li>
                      <li>Accessing KENCLUST Remotely</li>
                      <li>Datasets and File Formats – WGET command, and SRA Tools. Fasta, Fastq, SAM and BAM file
                        formats.</li>
                      <li>Human reference genome (hg18, hg19, hg38), indexing the reference genome.</li>
                      <li>NGS Data QC – Viewing reads, rimming ends, adapter removal, contaminant filtering, Kmer
                        masking, batch processing, sequence directory</li>
                      <li>Read Alignment – BWA commands, PE reads in interleaved and paired files, SE reads alignment,
                        Alignment of short and long genomes, SAI to SAM to BAM, sorting and indexing SAM files,
                        duplicate marking, local realignment, base quality score recalibration (BQSR)</li>
                      <li>Variant calling and annotation – HaplotypeCaller, Unified Genotyper, SNP and Indels annotation
                      </li>
                      <li>RNASEQ & Somatic SNPs, Indels and SNVs</li>
                      <li>Germline and somatic CNVs and SVs</li>
                    </ul>



                    <!-- PREREQUISITES AND FEES -->
                    <h4 class="myh4">Prerequisite and fees</h4>
                    <p>This training is ideal for people in biological, medical, pharma, and biotechnology industries
                      who wish to gain skills on how to detect genetic variations associated with disease from raw
                      sequenced data.</p>


                    <ul>
                      <li>The fee is Kshs. 85,000/ for East African residents. Non-East African students pay 850 USD.
                      </li>
                      <li>Fees covers tuition, training material, and certificate only.</li>
                      <li>
                        Fees are payable strictly in advance (at least 4 working days before the starting date.
                      </li>
                      <li>
                        Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                        least 2 weeks before the start) will be offered a 5% discount</li>
                      <li>There will be a 10% administrative charge for cancellations received in writing up to 20
                        working days before the start of the course. No refunds will be made for cancellations received
                        within 20 working days of the course start date or for the inability to attend the course for
                        whatever reasons. Substitutions may, of course, be made at any time, providing you inform us in
                        writing. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                        other reason that might lead to a delegate not being able to attend the course</li>
                      <li>Fee is payable by direct deposit, bankers cheque or MPESA to: Bioinformatics Institute of
                        Kenya Ltd. A/C No. 0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya.
                        The Paybill number for payments via MPESA is 400222 and the account number is 1679821#NGSDA.
                        Contact us for online payments.</li>
                    </ul>

                    <!-- ENROLLMENT DATES -->
                    <h4 class="myh4">Enrollment dates</h4>
                    <p>Every second and third weeks of the month</p>
                    </li>






                    <!-- COURSE NUMBER FOUR -->
                    <!-- Metagenomics -->
                    <li>
                      <!-- COURSE NAME -->
                      <h2>Metagenomics</h2>

                      <!-- ABOUT THE COURSE -->
                      <h4 class="myh4">About The Course</h4>
                      <p>This hands-on training will equip participants with knowledge and skills on how to generate,
                        manipulate, and analyze 16s rRNA, 18S rRNA, and ITS sequencing data. The objective is to enhance
                        the researchers’ capacity to conduct world class scientific research in microbial genetics and
                        boost their institution’s profile as a leading diagnostic and scientific center.
                      </p>



                      <!-- RESOURCES... -->
                      <h4 class="myh4">Resources</h4>
                      <p>Using the most powerful software, you will gain the latest skills in metagenomic data analysis.
                        Resources highlighted with an asterisk are a must have.
                      </p>
                      <ul>
                        <li>
                          <i>Web Resources - </i>
                          <ul>
                            <li><a href="https://www.ncbi.nlm.nih.gov/guide/all/">NCBI Tools</a></li>
                            <li><a href="https://www.arb-silva.de/">SILVA Database</a></li>
                          </ul>



                        <li>

                          <i>Software -</i>
                          <ul>
                            <li>MOTHUR V. 1.39,</li>
                            <li>R</li>
                            <li>Excel</li>
                            <li> Adobe Suite</li>
                            <li>QIIME</li>

                          </ul>


                        <li>
                          <i>Computing environment </i>


                          <ul>
                            <li>Linux computing using the KIBs 5DLAB Remote Cluster</li>
                          </ul>


                          <i>
                            Software add-ons for the participant</i>
                          <ul>
                            <li>PUTTY</li>
                            <li>WinSCP. Optional – R</li>
                            <li>XMING</li>
                            <li>BioLinux</li>
                            <li>Java Virtual Box</li>
                          </ul>
                        </li>

                    </li>

                    </li>
                    </ul>


                    <!-- TRAINING PROGRAMME: -->
                    <h4 class="myh4">Training program</h4>
                    <p>This is an 8-day intensive and hands-on training program. Training hours and days are flexible
                      and will be customized to fit each participant’s schedule. The duration of training is typically
                      one month but may be shorter or longer depending on each participant’s availability and
                      commitment. Training is mostly dry-lab based and practical hands-on with some few theoretical
                      lessons. Online classes can be arranged.:</p>
                    <ul>
                      <i>TOPICS & ACTIVITIES</i>
                      <li>Software installation</li>
                      <li>
                        Definition and scope of metagenomics, metagenomics state of the art, metagenomics applications
                      </li>
                      <li>
                        NGS Techniques: Illumina sequencing, Roche 454 sequencing. Ion Torrent: Proton / PGM sequencing,
                        PacBio sequencing, Nanopore Sequencing</li>
                      <li>
                        Microbial Identification methods: 16S rRNA, 18s rRNA , ITS, and shotgun gene sequencing</li>
                      <li>
                        Community profiling: alpha diversity, beta diversity,</li>
                      <li>Tests of significance, p-values, Tstatistics, and FDR values; T-test, ANOVA, Mann-Whitney,
                        PCOA and NMDS clustering, Random Forest</li>
                      <li>Classification, PERMANOVA, ANOSIM, HOMOVA, AMOVA, metastats, Lefse, and PERMDISP statistics
                      </li>
                      <li>Introduction to the Linux environment on a Windows machine via JVM, Remote computing using SSH
                        and Putty</li>
                      <li>Mastering Linux commands</li>
                      <li>Introduction to MOTHUR and QIIME</li>
                      <li>Creation of contigs from raw reads
                      </li>
                      <li>Data Pre-processing – Quality trimming, Deduplication of sequences , Removal of potential
                        redundancies arising from trimmed ends ,Pre-clustering to further remove any noise , Sorting by
                        abundance from most abundant to least abundant, Removal of chimeras using the VSEARCH algorithm,
                        Lineage removal, Assessment of error rates using a mock community</li>
                      <li>Clustering reads into OTUs, Making shared file, Classification of OTUS, Making taxonomy and
                        phylogenetic files</li>
                      <li>OTU Analysis – Alpha analysis: rarefaction, Beta analysis : PCOA and NMDS plots, AMOVA,
                        HOMOVA, Spearman correlation, UniFrac weighted, Metastats, Lefse,</li>
                      <li>Introduction to R
                        Analysis and Graphging using R – Stacked bar/area plot, Interactive pie chart, Rarefaction
                        curves, Phylogenetic tree, Heat tree analysis, FB Ratio, Chao 1 index, ACE index, Shannon index,
                        Simpson index, Fisher index, T-test/ANOVA p-value & T-statistic Mann-Whitney/Kruskal-Wallis
                        pvalue & Mann-Whitney statistic, PCOA plot, PERMANOVA, ANOSIM, PERMDISP statistics, Results</li>
                      <li>Tests of significance, p-values, Tstatistics, and FDR values; T-test, ANOVA, Mann-Whitney,
                        PCOA and NMDS clustering, Random Forest</li>
                      <li>table, Heatmap image, Hierarchical Clustering & Heatmap Visualization, Tree based on
                        unweighted Unifrac distance, SparCC correlation tables, network, and boxplots, T-test/ANOVA
                        p-value & T-statistic Mann-Whitney/Kruskal-Wallis pvalue & Mann-Whitney statistic, Zero inflated
                        Gaussian fit: p-values and FDR values, log2FC, logCPM, P-values, and FDR values, LDA scores,
                        p-values, FDR values, analysis results table, MDA Accuracy, OOB error rates, RF plots, important
                        features</li>
                      <li>Putting it all Together</li>

                      </li>
                    </ul>



                    <h4 class="myh4">Prerequisite and fees</h4>
                    <p>This training is ideal for people in biological, medical, pharma, and biotechnology industries
                      who wish to gain skills on how to detect genetic variations associated with disease from raw
                      sequenced data.</p>


                    <ul>
                      <li>The fee is Kshs. 85,000/ for East African residents. Non-East African students pay 850 USD.
                      </li>
                      <li>Fees covers tuition, training material, and certificate only.</li>
                      <li>
                        Fees are payable strictly in advance (at least 4 working days before the starting date.
                      </li>
                      <li>
                        Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                        least 2 weeks before the start) will be offered a 5% discount</li>
                      <li>There will be a 10% administrative charge for cancellations received in writing up to 20
                        working days before the start of the course. No refunds will be made for cancellations received
                        within 20 working days of the course start date or for the inability to attend the course for
                        whatever reasons. Substitutions may, of course, be made at any time, providing you inform us in
                        writing. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                        other reason that might lead to a delegate not being able to attend the course</li>
                      <li>Fee is payable by direct deposit, bankers cheque or MPESA to: Bioinformatics Institute of
                        Kenya Ltd. A/C No. 0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya.
                        The Paybill number for payments via MPESA is 400222 and the account number is 1679821#NGSDA.
                        Contact us for online payments.</li>
                    </ul>

                    <!-- ENROLLMENT DATES -->
                    <h4 class="myh4">Enrollment dates</h4>
                    <p>Every second and third weeks of the month</p>
                    </li>
                    <ul>
                      <li>
                        <i>High performance computing cluster - </i>
                        <ul>
                          <li>KENCLUST</li>
                        </ul>
                      </li>


                      <li>
                        <i>Datasets</i>
                        <ul>
                          <li>SRA Datasets</li>
                          <li>1,000 Genomes Project</li>
                          <li>The Cancer Genome Atlas (TCGA)</li>
                        </ul>
                      </li>

                      </li>

                      </li>
                    </ul>


                    <!-- TRAINING PROGRAMME: -->
                    <h4 class="myh4">Training program</h4>
                    <p>This is an intensive training program covering the following topics:</p>
                    <ul>
                      <li>NestGen Sequencing (NGS) principles, methods and technologies</li>
                      <li>NGS in Clinical Practice: Applications, Challenges and Promises</li>
                      <li>Introduction to the Linux OS and basic commands</li>
                      <li>SSH, Telnet, FTP, WinSCP, Putty, XMING, port forwarding and remote access</li>
                      <li>Accessing KENCLUST Remotely</li>
                      <li>Datasets and File Formats – WGET command, and SRA Tools. Fasta, Fastq, SAM and BAM file
                        formats.</li>
                      <li>Human reference genome (hg18, hg19, hg38), indexing the reference genome.</li>
                      <li>NGS Data QC – Viewing reads, rimming ends, adapter removal, contaminant filtering, Kmer
                        masking, batch processing, sequence directory</li>
                      <li>Read Alignment – BWA commands, PE reads in interleaved and paired files, SE reads alignment,
                        Alignment of short and long genomes, SAI to SAM to BAM, sorting and indexing SAM files,
                        duplicate marking, local realignment, base quality score recalibration (BQSR)</li>
                      <li>Variant calling and annotation – HaplotypeCaller, Unified Genotyper, SNP and Indels annotation
                      </li>
                      <li>RNASEQ & Somatic SNPs, Indels and SNVs</li>
                      <li>Germline and somatic CNVs and SVs</li>
                    </ul>



                    <!-- PREREQUISITES AND FEES -->
                    <h4 class="myh4">Prerequisite and fees</h4>
                    <p>This training is ideal for people in biological, medical, pharma, and biotechnology industries
                      who wish to gain skills on how to detect genetic variations associated with disease from raw
                      sequenced data.</p>


                    <ul>
                      <li>The fee is Kshs. 85,000/ for East African residents. Non-East African students pay 850 USD.
                      </li>
                      <li>Fees covers tuition, training material, and certificate only.</li>
                      <li>
                        Fees are payable strictly in advance (at least 4 working days before the starting date.
                      </li>
                      <li>
                        Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                        least 2 weeks before the start) will be offered a 5% discount</li>
                      <li>There will be a 10% administrative charge for cancellations received in writing up to 20
                        working days before the start of the course. No refunds will be made for cancellations received
                        within 20 working days of the course start date or for the inability to attend the course for
                        whatever reasons. Substitutions may, of course, be made at any time, providing you inform us in
                        writing. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                        other reason that might lead to a delegate not being able to attend the course</li>
                      <li>Fee is payable by direct deposit, bankers cheque or MPESA to: Bioinformatics Institute of
                        Kenya Ltd. A/C No. 0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya.
                        The Paybill number for payments via MPESA is 400222 and the account number is 1679821#NGSDA.
                        Contact us for online payments.</li>
                    </ul>

                    <!-- ENROLLMENT DATES -->
                    <h4 class="myh4">Enrollment dates</h4>
                    <p> The first week of every month.</p>
                    </li>





                    <!-- COURSE NUMBER FOUR -->
                    <!-- NGS Metabolomics -->
                    <li>
                      <h2>Metabolomics</h2>

                      <!-- ABOUT THE COURSE -->
                      <h4 class="myh4">About The Course</h4>
                      <p>Metabolomics is the fastest growing area of life sciences and involves the systematic study of
                        the small molecular metabolites in a cell, tissue, biofluid or cell culture media that are the
                        tangible result of cellular processes or responses to an environmental stress. The main goal of
                        metabolomics is to identify a list of differentially regulated metabolites for use in biomarker
                        discovery, pathway analysis, model construction, and scientific literature. The objective of the
                        Metabolomics Course is to acquaint the learner with emerging trends in metabolomics as well as
                        equip him with skills for generating and interrogating metabolomic data. The Metabolomics Course
                        is ideal for students and researchers working in the pharma, biotech, food, and agricultural
                        industries.
                      </p>

                      <!-- AT THE END OF THE TRAINING, THE PARTICIPANT SHOULD BE ABLE TO: -->
                      <p>
                        In this 5-day Metabolomics Course, students receive hands-on training on:
                      </p>
                      <!-- LIST OF DELIVERABLES PARTICIPANT SHOULD HAVE AT THE END OF THE TRAINING PROGRAMME. -->
                      <ul>
                        <li>Experimental design of metabolomics studies</li>
                        <li>Sample preparation including extraction, chromatography, and ionization efficiency
                          maximization</li>
                        <li>Generation of raw metabolomic LC/MS or GC/MS data</li>
                        <li>
                          Pre-processing data analysis including conversion of raw data to the mXMZL format, feature
                          detection, retention time correction and statistical analysis</li>
                        <li>Metabolite identification</li>
                        <li>Hypothesis testing</li>
                        <li>Experimental validation</li>
                      </ul>


                      <!-- RESOURCES... -->
                      <h4 class="myh4">Resources</h4>
                      <p>Using the most powerful software, you will gain the latest skills in metabolomics data
                        analysis.
                      </p>
                      <ul>
                        <li>
                          <i>Web Resources - </i>
                          <ul>
                            <li><a href="https://www.ncbi.nlm.nih.gov/guide/all/">NCBI Tools</a></li>
                            <li><a href="https://www.genome.gov/genetics-glossary/hapmap">HapMap</a></li>
                            <li><a href="https://www.cog-genomics.org/plink/">PLINK</a></li>
                            <li><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3415556/">SKAT-O</a></li>
                            <li><a href="">HMDB</a></li>
                            <li><a href="">MetaboAnalyst</a></li>
                            <li><a href="">Metabolights</a></li>
                            <li><a href="">OmicTools</a></li>
                            <li><a href="">XCMS(2)</a></li>
                            <li><a href="">OpenMS</a></li>
                            <li><a href="">MVAPACK</a></li>
                            <li><a href="">PubChem</a></li>
                            <li><a href="">PubChem</a></li>
                            <li><a href="">METLIN</a></li>
                            <li><a href="">KEGG</a></li>
                            <li><a href="">Metabo Miner</a></li>
                            <li><a href="">Metabolomic Analysis and Visualization Engine (MAVEN)</a></li>
                            <li><a href="">MolFind</a></li>
                            <li><a href="">COLMAR</a></li>
                            <li><a href="">MeltDB</a></li>
                          </ul>

                        </li>

                        <li>

                          <i>Software -</i>
                          <ul>
                            <li>Proteowizard</li>
                            <li>R</li>
                            <li>Bioconductor</li>
                            <li>XCMS</li>
                            <li>AMDIS</li>
                            <li>MATLAB</li>
                            <li>TopSpin</li>
                            <li>rNMR</li>
                            <li>batchcorr</li>
                            <li>Metabnorm</li>
                          </ul>

                        </li>


                        <li>

                          <i>HPC -</i>
                          <ul>
                            <li>KENCLUST</li>
                          </ul>

                        </li>


                        <li>
                          <i>Datasets</i>
                          <ul>
                            <li>HMDB</li>
                            <li>METLIN</li>
                            <li>PubChem</li>
                            <li>Metabolights</li>
                          </ul>
                        </li>

                    </li>
                    </ul>


                    <!-- TRAINING PROGRAMME: -->
                    <h4 class="myh4">Training program</h4>
                    <p>This is a 5-day course held on the fourth week of every month.</p>
                    <ul>
                      <li>Definition, Scope and Applications of Metabolomics</li>
                      <li>Applications of metabolomics in diagnostics, pharmaceutical research, systems biology,
                        nutrition, modelling, agricultural research, toxicology, and in the food industry</li>
                      <li>Bioinformatics Tools used in Metabolomics</li>

                      <li>Metabolomics Workflow – Experimental design, sample preparation, sample analysis, statistical
                        analysis</li>
                      <li>Exercises</li>
                      <li>Experimental Workflow – Metabolite Identification, Hypothesis Testing, Experimental
                        Validation, Exercises and Discussion</li>

                      <li>Case studies and discussions</li>

                    </ul>



                    <!-- PREREQUISITES AND FEES -->
                    <h4 class="myh4">Prerequisite and fees</h4>
                    <p>This training is ideal for people in biological, medical, pharma, and biotechnology industries
                      who wish to gain skills on how to accelerate drug discovery, identify disease biomarkers,
                      personalize medicine, detect metabolic conditions, conduct toxicology studies, select optimal
                      cultivars, among others.</p>
                    <p> Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                      least 2 weeks before the start) will be offered a 5% discount
                      There will be a 10% administrative charge for cancellations received in writing up to 20 working
                      days before the start of the course. No refunds will be made for cancellations received within 20
                      working days of the course start date or for the inability to attend the course for whatever
                      reasons. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                      other reason that might lead to a delegate not being able to attend the course. Substitutions may,
                      of course, be made at any time, providing you inform us in writing.</p>
                    <p>Fee is payable by bankers cheque or MPESA to: Bioinformatics Institute of Kenya Ltd. A/C No.
                      0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya. The Paybill number
                      for payments via Mpesa is 400222 and the account number is 1679821#your name. Contact us for
                      online payments</p>



                    <ul>
                      <li>The fee is Kshs. 85,000/- (850 USD, Inclusive of VAT 16%) for East African residents. Non-East
                        African students pay 25% more on all charges.
                      </li>
                      <li>
                        Fees covers tuition, training material, and certificate only.</li>

                      <li>
                        Fees are payable strictly in advance (at least 4 working days before the starting date.

                      </li>
                      <li>
                        Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                        least 2 weeks before the start) will be offered a 5% discount</li>

                      <li>There will be a 10% administrative charge for cancellations received in writing up to 20
                        working days before the start of the course. No refunds will be made for cancellations received
                        within 20 working days of the course start date or for the inability to attend the course for
                        whatever reasons. Substitutions may, of course, be made at any time, providing you inform us in
                        writing. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                        other reason that might lead to a delegate not being able to attend the course.</li>

                      <li>Fee is payable by direct deposit, bankers cheque or MPESA to: Bioinformatics Institute of
                        Kenya Ltd. A/C No. 0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya.
                        The Paybill number for payments via MPESA is 400222 and the account number is 1679821#NGSDA.
                        Contact us for online payments.</li>

                    </ul>

                    <!-- ENROLLMENT DATES -->
                    <h4 class="myh4">Enrollment dates</h4>
                    <p>The fourth week of every month.</p>
                    </li>












                    <!-- COURSE NUMBER FIVE -->
                    <!-- Computer-Aided Drug Design -->
                    <li>
                      <!-- COURSE NAME -->
                      <h2>Computer-Aided Drug Design</h2>

                      <!-- ABOUT THE COURSE -->
                      <h4 class="myh4">About The Course</h4>
                      <p>In this intensive training, participants will learn the science involved in disease target
                        identification, virtual screening techniques, in-silico generation of ligands, protein
                        optimization & energy minimization, molecular docking, creation of Grid Parameter & Dock
                        Parameter files, running the Docking Algorithm, pharmacophore modeling, and drug repurposing.
                        They will also learn how to select potent inhibitors on the basis of binding energies and
                        Lipinski ’s Rule of 5, how to look for H-bonds between ligands and active sites of protein
                        residues as well as aspects of drug Likeness, ADME & Toxicity. Practical hands-on training will
                        encompass training on the use of KenClust, software such as MarvinSketch, UCSF Chimera, AutoDock
                        Tools, Ligand Scout, Rosetta, Open Babel, and SPDV among others. Additional training will
                        include pharmacophore modelling and use of AI tools such as AlphaFold to predict 3D Structure
                        and generative AI for drug discovery. Lab work will include bacterial culture, antimicrobial
                        susceptibility testing, and genotoxicity testing.
                      </p>



                      <!-- RESOURCES... -->
                      <h4 class="myh4">Resources</h4>
                      <p>Using the most powerful software, you will gain the latest skills in metagenomic data analysis.
                        Resources highlighted with an asterisk are a must have.
                      </p>
                      <ul>
                        <li>
                          <i>Web Resources - </i>
                          <ul>
                            <li><a href="https://www.ncbi.nlm.nih.gov/guide/all/">NCBI Tools</a></li>
                            <li><a href="https://www.arb-silva.de/">SILVA Database</a></li>
                          </ul>



                        <li>

                          <i>Software -</i>
                          <ul>
                            <li>MOTHUR V. 1.39,</li>
                            <li>R</li>
                            <li>Excel</li>
                            <li> Adobe Suite</li>
                            <li>QIIME</li>

                          </ul>


                        <li>
                          <i>Computing environment </i>


                          <ul>
                            <li>Linux computing using the KIBs 5DLAB Remote Cluster</li>
                          </ul>


                          <i>
                            Software add-ons for the participant</i>
                          <ul>
                            <li>PUTTY</li>
                            <li>WinSCP. Optional – R</li>
                            <li>XMING</li>
                            <li>BioLinux</li>
                            <li>Java Virtual Box</li>
                          </ul>
                        </li>

                    </li>

                    </li>
                    </ul>


                    <!-- TRAINING PROGRAMME: -->
                    <h4 class="myh4">Training program</h4>
                    <p>This is an 8-day intensive and hands-on training program. Training hours and days are flexible
                      and will be customized to fit each participant’s schedule. The duration of training is typically
                      one month but may be shorter or longer depending on each participant’s availability and
                      commitment. Training is mostly dry-lab based and practical hands-on with some few theoretical
                      lessons. Online classes can be arranged.:</p>
                    <ul>
                      <i>TOPICS & ACTIVITIES</i>
                      <li>Software installation</li>
                      <li>
                        Definition and scope of metagenomics, metagenomics state of the art, metagenomics applications
                      </li>
                      <li>
                        NGS Techniques: Illumina sequencing, Roche 454 sequencing. Ion Torrent: Proton / PGM sequencing,
                        PacBio sequencing, Nanopore Sequencing</li>
                      <li>
                        Microbial Identification methods: 16S rRNA, 18s rRNA , ITS, and shotgun gene sequencing</li>
                      <li>
                        Community profiling: alpha diversity, beta diversity,</li>
                      <li>Tests of significance, p-values, Tstatistics, and FDR values; T-test, ANOVA, Mann-Whitney,
                        PCOA and NMDS clustering, Random Forest</li>
                      <li>Classification, PERMANOVA, ANOSIM, HOMOVA, AMOVA, metastats, Lefse, and PERMDISP statistics
                      </li>
                      <li>Introduction to the Linux environment on a Windows machine via JVM, Remote computing using SSH
                        and Putty</li>
                      <li>Mastering Linux commands</li>
                      <li>Introduction to MOTHUR and QIIME</li>
                      <li>Creation of contigs from raw reads
                      </li>
                      <li>Data Pre-processing – Quality trimming, Deduplication of sequences , Removal of potential
                        redundancies arising from trimmed ends ,Pre-clustering to further remove any noise , Sorting by
                        abundance from most abundant to least abundant, Removal of chimeras using the VSEARCH algorithm,
                        Lineage removal, Assessment of error rates using a mock community</li>
                      <li>Clustering reads into OTUs, Making shared file, Classification of OTUS, Making taxonomy and
                        phylogenetic files</li>
                      <li>OTU Analysis – Alpha analysis: rarefaction, Beta analysis : PCOA and NMDS plots, AMOVA,
                        HOMOVA, Spearman correlation, UniFrac weighted, Metastats, Lefse,</li>
                      <li>Introduction to R
                        Analysis and Graphging using R – Stacked bar/area plot, Interactive pie chart, Rarefaction
                        curves, Phylogenetic tree, Heat tree analysis, FB Ratio, Chao 1 index, ACE index, Shannon index,
                        Simpson index, Fisher index, T-test/ANOVA p-value & T-statistic Mann-Whitney/Kruskal-Wallis
                        pvalue & Mann-Whitney statistic, PCOA plot, PERMANOVA, ANOSIM, PERMDISP statistics, Results</li>
                      <li>Tests of significance, p-values, Tstatistics, and FDR values; T-test, ANOVA, Mann-Whitney,
                        PCOA and NMDS clustering, Random Forest</li>
                      <li>table, Heatmap image, Hierarchical Clustering & Heatmap Visualization, Tree based on
                        unweighted Unifrac distance, SparCC correlation tables, network, and boxplots, T-test/ANOVA
                        p-value & T-statistic Mann-Whitney/Kruskal-Wallis pvalue & Mann-Whitney statistic, Zero inflated
                        Gaussian fit: p-values and FDR values, log2FC, logCPM, P-values, and FDR values, LDA scores,
                        p-values, FDR values, analysis results table, MDA Accuracy, OOB error rates, RF plots, important
                        features</li>
                      <li>Putting it all Together</li>

                      </li>
                    </ul>



                    <h4 class="myh4">Prerequisite and fees</h4>
                    <p>This training is ideal for people in biological, medical, pharma, and biotechnology industries
                      who wish to gain skills on how to detect genetic variations associated with disease from raw
                      sequenced data.</p>


                    <ul>
                      <li>The fee is Kshs. 85,000/ for East African residents. Non-East African students pay 850 USD.
                      </li>
                      <li>Fees covers tuition, training material, and certificate only.</li>
                      <li>
                        Fees are payable strictly in advance (at least 4 working days before the starting date.
                      </li>
                      <li>
                        Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                        least 2 weeks before the start) will be offered a 5% discount</li>
                      <li>There will be a 10% administrative charge for cancellations received in writing up to 20
                        working days before the start of the course. No refunds will be made for cancellations received
                        within 20 working days of the course start date or for the inability to attend the course for
                        whatever reasons. Substitutions may, of course, be made at any time, providing you inform us in
                        writing. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                        other reason that might lead to a delegate not being able to attend the course</li>
                      <li>Fee is payable by direct deposit, bankers cheque or MPESA to: Bioinformatics Institute of
                        Kenya Ltd. A/C No. 0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya.
                        The Paybill number for payments via MPESA is 400222 and the account number is 1679821#NGSDA.
                        Contact us for online payments.</li>
                    </ul>

                    <!-- ENROLLMENT DATES -->
                    <h4 class="myh4">Enrollment dates</h4>
                    <p>Every second and third weeks of the month</p>
                    </li>
                    <ul>
                      <li>
                        <i>High performance computing cluster - </i>
                        <ul>
                          <li>KENCLUST</li>
                        </ul>
                      </li>


                      <li>
                        <i>Datasets</i>
                        <ul>
                          <li>SRA Datasets</li>
                          <li>1,000 Genomes Project</li>
                          <li>The Cancer Genome Atlas (TCGA)</li>
                        </ul>
                      </li>

                      </li>

                      </li>
                    </ul>


                    <!-- TRAINING PROGRAMME: -->
                    <h4 class="myh4">Training program</h4>
                    <p>This is an intensive training program covering the following topics:</p>
                    <ul>
                      <li>NestGen Sequencing (NGS) principles, methods and technologies</li>
                      <li>NGS in Clinical Practice: Applications, Challenges and Promises</li>
                      <li>Introduction to the Linux OS and basic commands</li>
                      <li>SSH, Telnet, FTP, WinSCP, Putty, XMING, port forwarding and remote access</li>
                      <li>Accessing KENCLUST Remotely</li>
                      <li>Datasets and File Formats – WGET command, and SRA Tools. Fasta, Fastq, SAM and BAM file
                        formats.</li>
                      <li>Human reference genome (hg18, hg19, hg38), indexing the reference genome.</li>
                      <li>NGS Data QC – Viewing reads, rimming ends, adapter removal, contaminant filtering, Kmer
                        masking, batch processing, sequence directory</li>
                      <li>Read Alignment – BWA commands, PE reads in interleaved and paired files, SE reads alignment,
                        Alignment of short and long genomes, SAI to SAM to BAM, sorting and indexing SAM files,
                        duplicate marking, local realignment, base quality score recalibration (BQSR)</li>
                      <li>Variant calling and annotation – HaplotypeCaller, Unified Genotyper, SNP and Indels annotation
                      </li>
                      <li>RNASEQ & Somatic SNPs, Indels and SNVs</li>
                      <li>Germline and somatic CNVs and SVs</li>
                    </ul>



                    <!-- PREREQUISITES AND FEES -->
                    <h4 class="myh4">Prerequisite and fees</h4>
                    <p>This training is ideal for people in biological, medical, pharma, and biotechnology industries
                      who wish to gain skills on how to detect genetic variations associated with disease from raw
                      sequenced data.</p>


                    <ul>
                      <li>The fee is Kshs. 85,000/ for East African residents. Non-East African students pay 850 USD.
                      </li>
                      <li>Fees covers tuition, training material, and certificate only.</li>
                      <li>
                        Fees are payable strictly in advance (at least 4 working days before the starting date.
                      </li>
                      <li>
                        Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                        least 2 weeks before the start) will be offered a 5% discount</li>
                      <li>There will be a 10% administrative charge for cancellations received in writing up to 20
                        working days before the start of the course. No refunds will be made for cancellations received
                        within 20 working days of the course start date or for the inability to attend the course for
                        whatever reasons. Substitutions may, of course, be made at any time, providing you inform us in
                        writing. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                        other reason that might lead to a delegate not being able to attend the course</li>
                      <li>Fee is payable by direct deposit, bankers cheque or MPESA to: Bioinformatics Institute of
                        Kenya Ltd. A/C No. 0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya.
                        The Paybill number for payments via MPESA is 400222 and the account number is 1679821#NGSDA.
                        Contact us for online payments.</li>
                    </ul>

                    <!-- ENROLLMENT DATES -->
                    <h4 class="myh4">Enrollment dates</h4>
                    <p> The first week of every month.</p>
                    </li>





                    <!-- COURSE NUMBER FOUR -->
                    <!-- NGS Metabolomics -->
                    <li>
                      <h2>Metabolomics</h2>

                      <!-- ABOUT THE COURSE -->
                      <h4 class="myh4">About The Course</h4>
                      <p>Metabolomics is the fastest growing area of life sciences and involves the systematic study of
                        the small molecular metabolites in a cell, tissue, biofluid or cell culture media that are the
                        tangible result of cellular processes or responses to an environmental stress. The main goal of
                        metabolomics is to identify a list of differentially regulated metabolites for use in biomarker
                        discovery, pathway analysis, model construction, and scientific literature. The objective of the
                        Metabolomics Course is to acquaint the learner with emerging trends in metabolomics as well as
                        equip him with skills for generating and interrogating metabolomic data. The Metabolomics Course
                        is ideal for students and researchers working in the pharma, biotech, food, and agricultural
                        industries.
                      </p>

                      <!-- AT THE END OF THE TRAINING, THE PARTICIPANT SHOULD BE ABLE TO: -->
                      <p>
                        In this 5-day Metabolomics Course, students receive hands-on training on:
                      </p>
                      <!-- LIST OF DELIVERABLES PARTICIPANT SHOULD HAVE AT THE END OF THE TRAINING PROGRAMME. -->
                      <ul>
                        <li>Experimental design of metabolomics studies</li>
                        <li>Sample preparation including extraction, chromatography, and ionization efficiency
                          maximization</li>
                        <li>Generation of raw metabolomic LC/MS or GC/MS data</li>
                        <li>
                          Pre-processing data analysis including conversion of raw data to the mXMZL format, feature
                          detection, retention time correction and statistical analysis</li>
                        <li>Metabolite identification</li>
                        <li>Hypothesis testing</li>
                        <li>Experimental validation</li>
                      </ul>


                      <!-- RESOURCES... -->
                      <h4 class="myh4">Resources</h4>
                      <p>Using the most powerful software, you will gain the latest skills in metabolomics data
                        analysis.
                      </p>
                      <ul>
                        <li>
                          <i>Web Resources - </i>
                          <ul>
                            <li><a href="https://www.ncbi.nlm.nih.gov/guide/all/">NCBI Tools</a></li>
                            <li><a href="https://www.genome.gov/genetics-glossary/hapmap">HapMap</a></li>
                            <li><a href="https://www.cog-genomics.org/plink/">PLINK</a></li>
                            <li><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3415556/">SKAT-O</a></li>
                            <li><a href="">HMDB</a></li>
                            <li><a href="">MetaboAnalyst</a></li>
                            <li><a href="">Metabolights</a></li>
                            <li><a href="">OmicTools</a></li>
                            <li><a href="">XCMS(2)</a></li>
                            <li><a href="">OpenMS</a></li>
                            <li><a href="">MVAPACK</a></li>
                            <li><a href="">PubChem</a></li>
                            <li><a href="">PubChem</a></li>
                            <li><a href="">METLIN</a></li>
                            <li><a href="">KEGG</a></li>
                            <li><a href="">Metabo Miner</a></li>
                            <li><a href="">Metabolomic Analysis and Visualization Engine (MAVEN)</a></li>
                            <li><a href="">MolFind</a></li>
                            <li><a href="">COLMAR</a></li>
                            <li><a href="">MeltDB</a></li>
                          </ul>

                        </li>

                        <li>

                          <i>Software -</i>
                          <ul>
                            <li>Proteowizard</li>
                            <li>R</li>
                            <li>Bioconductor</li>
                            <li>XCMS</li>
                            <li>AMDIS</li>
                            <li>MATLAB</li>
                            <li>TopSpin</li>
                            <li>rNMR</li>
                            <li>batchcorr</li>
                            <li>Metabnorm</li>
                          </ul>

                        </li>


                        <li>

                          <i>HPC -</i>
                          <ul>
                            <li>KENCLUST</li>
                          </ul>

                        </li>


                        <li>
                          <i>Datasets</i>
                          <ul>
                            <li>HMDB</li>
                            <li>METLIN</li>
                            <li>PubChem</li>
                            <li>Metabolights</li>
                          </ul>
                        </li>

                    </li>
                    </ul>


                    <!-- TRAINING PROGRAMME: -->
                    <h4 class="myh4">Training program</h4>
                    <p>This is a 5-day course held on the fourth week of every month.</p>
                    <ul>
                      <li>Definition, Scope and Applications of Metabolomics</li>
                      <li>Applications of metabolomics in diagnostics, pharmaceutical research, systems biology,
                        nutrition, modelling, agricultural research, toxicology, and in the food industry</li>
                      <li>Bioinformatics Tools used in Metabolomics</li>

                      <li>Metabolomics Workflow – Experimental design, sample preparation, sample analysis, statistical
                        analysis</li>
                      <li>Exercises</li>
                      <li>Experimental Workflow – Metabolite Identification, Hypothesis Testing, Experimental
                        Validation, Exercises and Discussion</li>

                      <li>Case studies and discussions</li>

                    </ul>



                    <!-- PREREQUISITES AND FEES -->
                    <h4 class="myh4">Prerequisite and fees</h4>
                    <p>This training is ideal for people in biological, medical, pharma, and biotechnology industries
                      who wish to gain skills on how to accelerate drug discovery, identify disease biomarkers,
                      personalize medicine, detect metabolic conditions, conduct toxicology studies, select optimal
                      cultivars, among others.</p>
                    <p> Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                      least 2 weeks before the start) will be offered a 5% discount
                      There will be a 10% administrative charge for cancellations received in writing up to 20 working
                      days before the start of the course. No refunds will be made for cancellations received within 20
                      working days of the course start date or for the inability to attend the course for whatever
                      reasons. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                      other reason that might lead to a delegate not being able to attend the course. Substitutions may,
                      of course, be made at any time, providing you inform us in writing.</p>
                    <p>Fee is payable by bankers cheque or MPESA to: Bioinformatics Institute of Kenya Ltd. A/C No.
                      0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya. The Paybill number
                      for payments via Mpesa is 400222 and the account number is 1679821#your name. Contact us for
                      online payments</p>



                    <ul>
                      <li>The fee is Kshs. 85,000/- (850 USD, Inclusive of VAT 16%) for East African residents. Non-East
                        African students pay 25% more on all charges.
                      </li>
                      <li>
                        Fees covers tuition, training material, and certificate only.</li>

                      <li>
                        Fees are payable strictly in advance (at least 4 working days before the starting date.

                      </li>
                      <li>
                        Groups with a minimum of 5 people will enjoy a 10% discount and early bird registration (at
                        least 2 weeks before the start) will be offered a 5% discount</li>

                      <li>There will be a 10% administrative charge for cancellations received in writing up to 20
                        working days before the start of the course. No refunds will be made for cancellations received
                        within 20 working days of the course start date or for the inability to attend the course for
                        whatever reasons. Substitutions may, of course, be made at any time, providing you inform us in
                        writing. KIBs is not liable for non-attendance due to travel disruptions, health problems or any
                        other reason that might lead to a delegate not being able to attend the course.</li>

                      <li>Fee is payable by direct deposit, bankers cheque or MPESA to: Bioinformatics Institute of
                        Kenya Ltd. A/C No. 0012342070001, Cooperative Bank of Kenya, Co-op House Branch, Nairobi, Kenya.
                        The Paybill number for payments via MPESA is 400222 and the account number is 1679821#NGSDA.
                        Contact us for online payments.</li>

                    </ul>

                    <!-- ENROLLMENT DATES -->
                    <h4 class="myh4">Enrollment dates</h4>
                    <p>The fourth week of every month.</p>
                    </li>
                  </ol>
                </section>
                <br><br><br><br><br><br><br><br><br><br>
              </div>
            </div>
          </div>
        </div>
    </div>
  </div>



  <div class="debug">
    <div class="container-fluid">
      <div class="row">
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
        <div class="col-1">
          <div class="col-content"></div>
        </div>
      </div>
    </div>
  </div>

  <script type='text/javascript'>
    const lazyloadRunObserver = () => {
      const lazyloadBackgrounds = document.querySelectorAll(`.e-con.e-parent:not(.e-lazyloaded)`);
      const lazyloadBackgroundObserver = new IntersectionObserver((entries) => {
        entries.forEach((entry) => {
          if (entry.isIntersecting) {
            let lazyloadBackground = entry.target;
            if (lazyloadBackground) {
              lazyloadBackground.classList.add('e-lazyloaded');
            }
            lazyloadBackgroundObserver.unobserve(entry.target);
          }
        });
      }, {
        rootMargin: '200px 0px 200px 0px'
      });
      lazyloadBackgrounds.forEach((lazyloadBackground) => {
        lazyloadBackgroundObserver.observe(lazyloadBackground);
      });
    };
    const events = [
      'DOMContentLoaded',
      'elementor/lazyload/observe',
    ];
    events.forEach((event) => {
      document.addEventListener(event, lazyloadRunObserver);
    });
  </script>
  <link rel='stylesheet' id='elementor-post-83187-css'
    href='../wp-content/uploads/elementor/css/post-83187.css%3Fver=1734206478.css' type='text/css' media='all' />
  <link rel='stylesheet' id='elementor-post-84149-css'
    href='../wp-content/uploads/elementor/css/post-84149.css%3Fver=1734377221.css' type='text/css' media='all' />
  <link rel='stylesheet' id='elementor-post-84206-css'
    href='../wp-content/uploads/elementor/css/post-84206.css%3Fver=1734377254.css' type='text/css' media='all' />
  <link rel='stylesheet' id='e-sticky-css'
    href='../wp-content/plugins/elementor-pro/assets/css/modules/sticky.min.css%3Fver=3.25.5.css' type='text/css'
    media='all' />
  <script type="text/javascript"
    src="https://cdnjs.cloudflare.com/ajax/libs/flickity/2.3.0/flickity.pkgd.min.js?ver=2.3.0" id="flickity-js-js">
  </script>
  <script type="text/javascript"
    src="../wp-content/plugins/elementor-pro/assets/lib/sticky/jquery.sticky.min.js%3Fver=3.25.5" id="e-sticky-js">
  </script>
  <script type="text/javascript" src="../wp-includes/js/imagesloaded.min.js%3Fver=5.0.0" id="imagesloaded-js">
  </script>
  <script type="text/javascript"
    src="../wp-content/plugins/elementor-pro/assets/lib/smartmenus/jquery.smartmenus.min.js%3Fver=1.2.1"
    id="smartmenus-js"></script>
  <script type="text/javascript"
    src="../wp-content/plugins/wp-smush-pro/app/assets/js/smush-lazy-load.min.js%3Fver=3.9.9" id="smush-lazy-load-js">
  </script>
  <script type="text/javascript" src="../wp-content/themes/wfx-blog-new-theme/assets/js/index.js%3Fver=1688463337"
    id="script-js"></script>
  <script type="text/javascript" src="https://js.chilipiper.com/marketing.js?ver=6.7.1" id="chilipiper-js"></script>
  <script type="text/javascript" id="wfxBlogAPI-js-extra">
    /* <![CDATA[ */
    var wfxBAJAX = {
      "apibase": "https:\/\/whatfix.com\/blog\/wp-json\/api\/v1\/base",
      "ajaxurl": "https:\/\/whatfix.com\/blog\/wp-admin\/admin-ajax.php",
      "security": "7ea511b35b",
      "restnonce": "651b97b607"
    };
    /* ]]> */
  </script>
  <script type="text/javascript" src="../wp-content/themes/wfx-blog-new-theme/assets/js/wfxBlogAPI.js%3Fver=1737095396"
    id="wfxBlogAPI-js">
  </script>
  <script type="text/javascript" src="https://whatfix.involve.me/embed?ver=6.7.1" id="wfx-involve-me-js"></script>
  <script type="text/javascript"
    src="../wp-content/plugins/elementor-pro/assets/js/webpack-pro.runtime.min.js%3Fver=3.25.5"
    id="elementor-pro-webpack-runtime-js"></script>
  <script type="text/javascript" src="../wp-content/plugins/elementor/assets/js/webpack.runtime.min.js%3Fver=3.25.11"
    id="elementor-webpack-runtime-js"></script>
  <script type="text/javascript" src="../wp-content/plugins/elementor/assets/js/frontend-modules.min.js%3Fver=3.25.11"
    id="elementor-frontend-modules-js"></script>
  <script type="text/javascript" src="../wp-includes/js/dist/hooks.min.js%3Fver=4d63a3d491d11ffd8ac6" id="wp-hooks-js">
  </script>
  <script type="text/javascript" src="../wp-includes/js/dist/i18n.min.js%3Fver=5e580eb46a90c2b997e6" id="wp-i18n-js">
  </script>
  <script type="text/javascript" id="wp-i18n-js-after">
    /* <![CDATA[ */
    wp.i18n.setLocaleData({
      'text direction\u0004ltr': ['ltr']
    });
    /* ]]> */
  </script>
  <script type="text/javascript" id="elementor-pro-frontend-js-before">
    /* <![CDATA[ */
    var ElementorProFrontendConfig = {
      "ajaxurl": "https:\/\/whatfix.com\/blog\/wp-admin\/admin-ajax.php",
      "nonce": "ec85e77a0c",
      "urls": {
        "assets": "https:\/\/whatfix.com\/blog\/wp-content\/plugins\/elementor-pro\/assets\/",
        "rest": "https:\/\/whatfix.com\/blog\/wp-json\/"
      },
      "settings": {
        "lazy_load_background_images": true
      },
      "popup": {
        "hasPopUps": true
      },
      "shareButtonsNetworks": {
        "facebook": {
          "title": "Facebook",
          "has_counter": true
        },
        "twitter": {
          "title": "Twitter"
        },
        "linkedin": {
          "title": "LinkedIn",
          "has_counter": true
        },
        "pinterest": {
          "title": "Pinterest",
          "has_counter": true
        },
        "reddit": {
          "title": "Reddit",
          "has_counter": true
        },
        "vk": {
          "title": "VK",
          "has_counter": true
        },
        "odnoklassniki": {
          "title": "OK",
          "has_counter": true
        },
        "tumblr": {
          "title": "Tumblr"
        },
        "digg": {
          "title": "Digg"
        },
        "skype": {
          "title": "Skype"
        },
        "stumbleupon": {
          "title": "StumbleUpon",
          "has_counter": true
        },
        "mix": {
          "title": "Mix"
        },
        "telegram": {
          "title": "Telegram"
        },
        "pocket": {
          "title": "Pocket",
          "has_counter": true
        },
        "xing": {
          "title": "XING",
          "has_counter": true
        },
        "whatsapp": {
          "title": "WhatsApp"
        },
        "email": {
          "title": "Email"
        },
        "print": {
          "title": "Print"
        },
        "x-twitter": {
          "title": "X"
        },
        "threads": {
          "title": "Threads"
        }
      },
      "facebook_sdk": {
        "lang": "en_US",
        "app_id": ""
      },
      "lottie": {
        "defaultAnimationUrl": "https:\/\/whatfix.com\/blog\/wp-content\/plugins\/elementor-pro\/modules\/lottie\/assets\/animations\/default.json"
      }
    };
    /* ]]> */
  </script>
  <script type="text/javascript" src="../wp-content/plugins/elementor-pro/assets/js/frontend.min.js%3Fver=3.25.5"
    id="elementor-pro-frontend-js"></script>
  <script type="text/javascript" src="../wp-includes/js/jquery/ui/core.min.js%3Fver=1.13.3" id="jquery-ui-core-js">
  </script>
  <script type="text/javascript" id="elementor-frontend-js-before">
    /* <![CDATA[ */
    var elementorFrontendConfig = {
      "environmentMode": {
        "edit": false,
        "wpPreview": false,
        "isScriptDebug": false
      },
      "i18n": {
        "shareOnFacebook": "Share on Facebook",
        "shareOnTwitter": "Share on Twitter",
        "pinIt": "Pin it",
        "download": "Download",
        "downloadImage": "Download image",
        "fullscreen": "Fullscreen",
        "zoom": "Zoom",
        "share": "Share",
        "playVideo": "Play Video",
        "previous": "Previous",
        "next": "Next",
        "close": "Close",
        "a11yCarouselWrapperAriaLabel": "Carousel | Horizontal scrolling: Arrow Left & Right",
        "a11yCarouselPrevSlideMessage": "Previous slide",
        "a11yCarouselNextSlideMessage": "Next slide",
        "a11yCarouselFirstSlideMessage": "This is the first slide",
        "a11yCarouselLastSlideMessage": "This is the last slide",
        "a11yCarouselPaginationBulletMessage": "Go to slide"
      },
      "is_rtl": false,
      "breakpoints": {
        "xs": 0,
        "sm": 480,
        "md": 768,
        "lg": 1025,
        "xl": 1440,
        "xxl": 1600
      },
      "responsive": {
        "breakpoints": {
          "mobile": {
            "label": "Mobile Portrait",
            "value": 767,
            "default_value": 767,
            "direction": "max",
            "is_enabled": true
          },
          "mobile_extra": {
            "label": "Mobile Landscape",
            "value": 880,
            "default_value": 880,
            "direction": "max",
            "is_enabled": false
          },
          "tablet": {
            "label": "Tablet Portrait",
            "value": 1024,
            "default_value": 1024,
            "direction": "max",
            "is_enabled": true
          },
          "tablet_extra": {
            "label": "Tablet Landscape",
            "value": 1200,
            "default_value": 1200,
            "direction": "max",
            "is_enabled": false
          },
          "laptop": {
            "label": "Laptop",
            "value": 1366,
            "default_value": 1366,
            "direction": "max",
            "is_enabled": false
          },
          "widescreen": {
            "label": "Widescreen",
            "value": 2400,
            "default_value": 2400,
            "direction": "min",
            "is_enabled": false
          }
        },
        "hasCustomBreakpoints": false
      },
      "version": "3.25.11",
      "is_static": false,
      "experimentalFeatures": {
        "e_font_icon_svg": true,
        "e_swiper_latest": true,
        "e_nested_atomic_repeaters": true,
        "e_optimized_control_loading": true,
        "e_onboarding": true,
        "e_css_smooth_scroll": true,
        "theme_builder_v2": true,
        "home_screen": true,
        "nested-elements": true,
        "editor_v2": true,
        "link-in-bio": true,
        "floating-buttons": true
      },
      "urls": {
        "assets": "https:\/\/whatfix.com\/blog\/wp-content\/plugins\/elementor\/assets\/",
        "ajaxurl": "https:\/\/whatfix.com\/blog\/wp-admin\/admin-ajax.php",
        "uploadUrl": "https:\/\/whatfix.com\/blog\/wp-content\/uploads"
      },
      "nonces": {
        "floatingButtonsClickTracking": "0c7b981212"
      },
      "swiperClass": "swiper",
      "settings": {
        "page": [],
        "editorPreferences": []
      },
      "kit": {
        "active_breakpoints": ["viewport_mobile", "viewport_tablet"],
        "global_image_lightbox": "yes",
        "lightbox_enable_counter": "yes",
        "lightbox_enable_fullscreen": "yes",
        "lightbox_enable_zoom": "yes",
        "lightbox_enable_share": "yes",
        "lightbox_title_src": "title",
        "lightbox_description_src": "description"
      },
      "post": {
        "id": 15822,
        "title": "14%20Types%20of%20Employee%20Training%20Programs%20%28%2BBenefits%2C%20Examples%29",
        "excerpt": "",
        "featuredImage": "https:\/\/whatfix.com\/blog\/wp-content\/uploads\/2023\/05\/Employee-Training.png"
      }
    };
    /* ]]> */
  </script>
  <script type="text/javascript" src="../wp-content/plugins/elementor/assets/js/frontend.min.js%3Fver=3.25.11"
    id="elementor-frontend-js"></script>
  <script type="text/javascript"
    src="../wp-content/plugins/elementor-pro/assets/js/elements-handlers.min.js%3Fver=3.25.5"
    id="pro-elements-handlers-js"></script>
  <script type="text/javascript">
    piAId = '674673';
    piCId = '1671';
    piHostname = 'pi.pardot.com';

    (function () {
      function async_load() {
        var s = document.createElement('script');
        s.type = 'text/javascript';
        s.src = ('https:' == document.location.protocol ? 'https://pi' : 'http://cdn') + '.pardot.com/pd.js';
        var c = document.getElementsByTagName('script')[0];
        c.parentNode.insertBefore(s, c);
      }
      if (window.attachEvent) {
        window.attachEvent('onload', async_load);
      } else {
        window.addEventListener('load', async_load, false);
      }
    })();
  </script>



  <footer>

    <div class="container my-5">


      <footer class="text-center text-lg-start text-white" style="background-color: #1c2331">

        <section class="d-flex justify-content-between p-4" style="background-color: #081B42">

          <div class="me-5">
            <span>
              <h2>Get connected with us on social networks:</h2>
            </span>
          </div>

          <div>
            <a href="" class="text-white me-4">
              <i class="fab fa-facebook-f"></i>
            </a>
            <a href="" class="text-white me-4">
              <i class="fab fa-twitter"></i>
            </a>
            <a href="" class="text-white me-4">
              <i class="fab fa-google"></i>
            </a>
            <a href="" class="text-white me-4">
              <i class="fab fa-instagram"></i>
            </a>
            <a href="" class="text-white me-4">
              <i class="fab fa-linkedin"></i>
            </a>
            <a href="" class="text-white me-4">
              <i class="fab fa-github"></i>
            </a>
          </div>

        </section>

        <section class="">
          <div class="container text-center text-md-start mt-5">

            <div class="row mt-3">

              <div class="col-md-3 col-lg-4 col-xl-3 mx-auto mb-4">

                <h6 class="text-uppercase fw-bold">
                  <svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="119px" height="85px"
                    style="shape-rendering:geometricPrecision; text-rendering:geometricPrecision; image-rendering:optimizeQuality; fill-rule:evenodd; clip-rule:evenodd"
                    xmlns:xlink="http://www.w3.org/1999/xlink">
                    <g>
                      <path style="opacity:0.285" fill="#325c2b"
                        d="M 19.5,14.5 C 20.2891,14.7828 20.9558,15.2828 21.5,16C 19.3996,24.4269 21.5663,31.5936 28,37.5C 28.6877,38.3317 28.521,38.9984 27.5,39.5C 16.4229,33.7115 13.7562,25.3782 19.5,14.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.908" fill="#335a2c"
                        d="M 51.5,35.5 C 55.4589,34.5778 57.7922,36.0778 58.5,40C 56.78,43.4649 54.1133,44.2982 50.5,42.5C 49.073,39.8845 49.4063,37.5511 51.5,35.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.907" fill="#335c2c"
                        d="M 59.5,35.5 C 61.8333,35.5 64.1667,35.5 66.5,35.5C 66.1796,38.8984 66.513,42.2317 67.5,45.5C 72.0589,43.8889 76.2256,44.5556 80,47.5C 85.2396,66.0948 78.4063,73.0948 59.5,68.5C 59.5,57.5 59.5,46.5 59.5,35.5 Z M 68.5,48.5 C 71.368,48.085 73.5347,49.085 75,51.5C 75.9777,55.9002 75.4777,60.0669 73.5,64C 67.6575,65.1706 65.3242,62.6706 66.5,56.5C 66.6205,53.5589 67.2872,50.8922 68.5,48.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.275" fill="#325d2c"
                        d="M 36.5,6.5 C 46.9026,14.8006 48.736,24.8006 42,36.5C 35.9588,42.7089 29.7921,48.7089 23.5,54.5C 25.5,55.1667 27.5,55.8333 29.5,56.5C 26.5,57.1667 23.5,57.8333 20.5,58.5C 19.9598,59.7481 19.2931,60.9148 18.5,62C 32.8547,63.3077 32.5213,64.141 17.5,64.5C 16.7588,68.7947 16.5921,73.128 17,77.5C 12.3788,69.2313 12.3788,60.898 17,52.5C 22.6771,45.6567 28.8438,39.3233 35.5,33.5C 33.1667,32.8333 30.8333,32.1667 28.5,31.5C 31.8333,30.8333 35.1667,30.1667 38.5,29.5C 39.2458,28.2638 39.5792,26.9305 39.5,25.5C 36.1016,25.8204 32.7683,25.487 29.5,24.5C 33.1073,23.5108 36.774,23.1774 40.5,23.5C 40.5,22.1667 40.5,20.8333 40.5,19.5C 24.5137,18.9959 24.5137,18.1626 40.5,17C 38.7396,13.6488 37.4063,10.1488 36.5,6.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.897" fill="#325c2d"
                        d="M 50.5,44.5 C 52.8333,44.5 55.1667,44.5 57.5,44.5C 57.5,52.5 57.5,60.5 57.5,68.5C 55.1667,68.5 52.8333,68.5 50.5,68.5C 50.5,60.5 50.5,52.5 50.5,44.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.889" fill="#325c2c"
                        d="M 87.5,44.5 C 92.2142,44.3531 96.8809,44.6864 101.5,45.5C 102.052,47.3307 102.052,48.9974 101.5,50.5C 97.4001,48.0246 93.4001,48.0246 89.5,50.5C 93.6423,52.39 97.809,54.39 102,56.5C 104.429,62.5457 102.596,66.7123 96.5,69C 92.141,69.5343 87.8076,69.3676 83.5,68.5C 83.5792,67.0695 83.2458,65.7362 82.5,64.5C 83,64.1667 83.5,63.8333 84,63.5C 87.4815,65.5917 90.9815,65.7584 94.5,64C 95.7472,62.5064 95.7472,61.0064 94.5,59.5C 91.5499,58.5123 88.5499,57.679 85.5,57C 82.6508,54.4406 81.8175,51.2739 83,47.5C 84.4087,46.2189 85.9087,45.2189 87.5,44.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.851" fill="#325b2c"
                        d="M 34.5,45.5 C 36.679,45.84 38.5123,46.84 40,48.5C 43.6939,54.5545 46.8606,60.8878 49.5,67.5C 46.9135,68.48 44.2469,68.8134 41.5,68.5C 40.9659,60.2348 38.6325,52.5681 34.5,45.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.395" fill="#237F99"
                        d="M 72.5,71.5 C 74.9509,73.478 77.6176,73.8114 80.5,72.5C 81.3081,72.8076 81.9747,73.3076 82.5,74C 81.3595,74.7412 81.3595,75.5745 82.5,76.5C 78.8352,77.2329 74.8352,77.5662 70.5,77.5C 70.67,76.1779 70.3366,75.0113 69.5,74C 70.2529,72.513 71.0862,72.3463 72,73.5C 72.4643,72.9056 72.631,72.2389 72.5,71.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.459" fill="#325d2c"
                        d="M 99.5,71.5 C 100.5,71.5 101.5,71.5 102.5,71.5C 102.5,73.5 102.5,75.5 102.5,77.5C 99.5525,77.7771 96.7191,77.4438 94,76.5C 93.1282,72.8651 94.2949,71.8651 97.5,73.5C 98.7133,73.2528 99.38,72.5862 99.5,71.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.507" fill="#325c2c"
                        d="M 49.5,71.5 C 53.0324,71.3789 55.8658,72.7122 58,75.5C 58.9356,73.2915 60.6023,71.9582 63,71.5C 64.4007,73.1922 66.2341,73.8588 68.5,73.5C 68.5,74.8333 68.5,76.1667 68.5,77.5C 67.1667,77.5 65.8333,77.5 64.5,77.5C 64.1667,73.5 63.8333,73.5 63.5,77.5C 60.319,77.2577 57.319,77.2577 54.5,77.5C 53.3756,72.2792 52.5423,72.2792 52,77.5C 51.4153,75.3386 50.5819,73.3386 49.5,71.5 Z" />
                    </g>
                    <g>
                      <path style="opacity:0.393" fill="#335d2b"
                        d="M 83.5,73.5 C 86.5,73.5 89.5,73.5 92.5,73.5C 92.5,74.8333 92.5,76.1667 92.5,77.5C 89.863,76.7678 87.363,76.7678 85,77.5C 83.8118,76.4366 83.3118,75.1033 83.5,73.5 Z" />
                    </g>
                  </svg>
                </h6>
                <hr class="mb-4 mt-0 d-inline-block mx-auto"
                  style="width: 60px; background-color: #7c4dff; height: 2px" />
                <p>
                  "AFRICA'S BIOTECH COMPANY"
                </p>
              </div>



              <div class="col-md-3 col-lg-2 col-xl-2 mx-auto mb-4">

                <h6 class="text-uppercase fw-bold">Useful links</h6>
                <hr class="mb-4 mt-0 d-inline-block mx-auto"
                  style="width: 60px; background-color: #7c4dff; height: 2px" />
                <p>
                  <a href="#!" class="text-white">Your Account</a>
                </p>
                <p>
                  <a href="#!" class="text-white">Become an Affiliate</a>
                </p>
                <p>
                  <a href="#!" class="text-white">Shipping Rates</a>
                </p>
                <p>
                  <a href="#!" class="text-white">Help</a>
                </p>
              </div>

              <div class="col-md-4 col-lg-3 col-xl-3 mx-auto mb-md-0 mb-4">

                <h6 class="text-uppercase fw-bold">Contact</h6>
                <hr class="mb-4 mt-0 d-inline-block mx-auto"
                  style="width: 60px; background-color: #7c4dff; height: 2px" />
                <p><i class="fas fa-home mr-3"></i> Kenbelt Park, Nairobi, Kenya</p>
                <p><i class="fas fa-envelope mr-3"></i> info@kibs.com</p>
                <p><i class="fas fa-phone mr-3"></i> +254 113 245 615</p>
                <p><i class="fas fa-print mr-3"></i> + 254 783 823 286</p>
              </div>

            </div>

          </div>
        </section>

        <div class="text-center p-3" style="background-color: rgba(0, 0, 0, 0.2)">
          © Copyright 2025. All Rights Reserved:

        </div>

      </footer>
    </div>
  </footer>








  <div class="accessibility-widget w-embed w-script">
    <div id="accessibility-widget">

      <div id="widget-button" role="button" tabindex="0" aria-label="Open Accessibility Options"
        onkeydown="handleButtonKey(event)">
        <img
          src="https://cdn.prod.website-files.com/600be113c4111047016de64a/66d6024aa7544dcc894d9c9d_Accessibility%20icon.svg"
          alt="Accessibility Icon">
      </div>
      <div id="settings-panel">
        <button id="increase-font">Zoom In</button>
        <button id="decrease-font">Zoom Out</button>
        <button id="toggle-contrast">Toggle Contrast</button>
      </div>
    </div>

    <style>
      #accessibility-widget {
        position: fixed;
        bottom: 70px;
        left: 15px;
        z-index: 1000;
      }

      #widget-button {
        background-color: #fff;
        width: 40px;
        height: 40px;
        border-radius: 50%;
        display: flex;
        align-items: center;
        justify-content: center;
        cursor: pointer;
        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.3);
      }

      #widget-button img {
        width: 24px;
        height: 24px;
      }

      #settings-panel {
        display: none;
        flex-direction: column;
        align-items: center;
        margin-top: 10px;
        background-color: #f1f1f1;
        border-radius: 5px;
        padding: 10px;
        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.3);
      }

      #settings-panel button {
        margin: 5px 0;
        padding: 5px 10px;
        font-size: 14px;
        cursor: pointer;
        width: 100%;
        background-color: #fff;
        color: #000;
        border: 1px solid #000;
        border-radius: 5px;
        box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2);
      }

      .high-contrast {
        background-color: #000 !important;
        color: #fff !important;
      }

      .high-contrast .Nav,
      .high-contrast .navlink {
        color: #fff !important;
        background-color: #000 !important;
      }

      /* Mobile-specific styles */
      @media (max-width: 767px) {
        #accessibility-widget {
          bottom: 20px;
          left: 10px;
        }

        #widget-button {
          width: 30px;
          height: 30px;
        }

        #widget-button img {
          width: 18px;
          height: 18px;
        }

        #settings-panel button {
          padding: 5px;
        }

        html {
          overflow-x: hidden;
        }
      }
    </style>

    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
      integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous">
    </script>
    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/umd/popper.min.js"
      integrity="sha384-I7E8VVD/ismYTF4hNIPjVp/Zjvgyol6VFvRkX/vR+Vc4jQkC+hVqc2pM8ODewa9r" crossorigin="anonymous">
    </script>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.min.js"
      integrity="sha384-0pUGZvbkm6XF6gxjEnlmuGrJXVbNuzT9qBBavbLwCsOGabYfZo0T0to5eqruptLy" crossorigin="anonymous">
    </script>

    <script>
      document.addEventListener("DOMContentLoaded", function () {
        const widgetButton = document.getElementById('widget-button');
        const settingsPanel = document.getElementById('settings-panel');
        let fontSize = 16;

        widgetButton.addEventListener('click', function () {
          if (settingsPanel.style.display === 'none' || settingsPanel.style.display === '') {
            settingsPanel.style.display = 'flex';
          } else {
            settingsPanel.style.display = 'none';
          }
        });

        document.getElementById("increase-font").addEventListener("click", function () {
          fontSize += 2;
          document.body.style.fontSize = fontSize + 'px';
        });

        document.getElementById("decrease-font").addEventListener("click", function () {
          fontSize -= 2;
          document.body.style.fontSize = fontSize + 'px';
        });

        document.getElementById("toggle-contrast").addEventListener("click", function () {
          document.body.classList.toggle('high-contrast');
          document.querySelector('.Nav').classList.toggle('high-contrast');
          document.querySelectorAll('.navlink').forEach(function (link) {
            link.classList.toggle('high-contrast');
          });
        });
      });
    </script>
  </div>
  <div class="cookie-script w-embed w-script">
    <script type="text/javascript" charset="UTF-8"
      src="http://cdn.cookie-script.com/s/d908b12cd9c87fea4284a4185b750faa.js"></script>
  </div>
  <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.5.1.min.dc5e7f18c8.js?site=600be113c4111047016de64a"
    type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous">
  </script>
  <script
    src="https://cdn.prod.website-files.com/600be113c4111047016de64a/js/webflow.0527a82382a637c47b9102f8ac6f9982.js"
    type="text/javascript"></script>
  <script>
    // PAGE TRANSITION
    function tr(myLink) {
      return (myLink.host == window.location.host);
    }
    $('a:not(.w-pagination-next)').each(function () {
      if (tr(this) && (this).href.indexOf('#') === -1) {
        $(this).click(function (e) {
          e.preventDefault();
          var moduleURL = jQuery(this).attr("href");
          setTimeout(function () {
            window.location = moduleURL
          }, 350);
          $('.tr').css("opacity", "1");
        });
      }
    });
    $(window).bind("pageshow", function (event) {
      if (event.originalEvent.persisted) {
        window.location.reload()
      }
    });

    // NAV INTERACTIONS
    $('.menu-btn').click(function () {
      $('.nav').toggleClass('nav-open');
    });

    // LOCK SCROLL - MENU OPEN
    (function () {
      $(".menu-btn").click(function () {
        $("body").toggleClass("overflow")
      })
    }).call(this),
      function () {
        var t = document.getElementById("menu"),
          e = null;
        t.addEventListener("touchstart", function (t) {
          1 === t.targetTouches.length && (e = t.targetTouches[0].clientY)
        }, !1), t.addEventListener("touchmove", function (n) {
          1 === n.targetTouches.length && function (n) {
            var o = n.targetTouches[0].clientY - e;
            0 === t.scrollTop && o > 0 && n.preventDefault(), t.scrollHeight - t.scrollTop <= t
              .clientHeight && o < 0 && n.preventDefault()
          }(n)
        }, !1)
      }();
  </script>


  <script>
    document.addEventListener('DOMContentLoaded', function () {
      // Select all interactive elements
      var focusableElements =
        'a[href], button, input, textarea, select, details, [tabindex]:not([tabindex="-1"])';
      var focusables = document.querySelectorAll(focusableElements);

      // Add a keydown event listener to each focusable element
      focusables.forEach(function (element) {
        element.addEventListener('keydown', function (e) {
          if (e.key === 'Tab') {
            // Prevent the default tab behavior
            e.preventDefault();
            var index = Array.prototype.indexOf.call(focusables, e.target);
            var nextElement = focusables[index + (e.shiftKey ? -1 : 1)];
            if (!nextElement) {
              nextElement = focusables[e.shiftKey ? focusables.length - 1 : 0];
            }
            nextElement.focus();
          }
        });
      });
    });
  </script>

  <script>
    // NOTIFICATION BAR CONDITIONAL VISIBILITY
    if ($(".notif").is(":visible")) {
      $(".nav-wrap").addClass('with-notif');
    } else {}
    $('.notif-close').click(function () {
      $('.nav-wrap').removeClass('with-notif');
    });
  </script>

</body>

</html>